feat(content): Patir 2 (#9959)
endless-sky/endless-sky: data/kahet/kahet missions.txt

Reviewer: I feel like the payment at the end of this chain is slightly low. Only 370,000 credits? I can earn that much in two days with the Remnant!

Maybe bump it to three million, especially to compensate for crew salaries?

Author: Changed to a million and a half, and added the rest of your reviews by hand

> # Copyright (c) 2024 by Tommaso Becca

Reviewer: Why are you changing this?

	"reputation: Ka'sei" = 1

event "patir mystery timer"

ship "Cloak Check"
	attributes
		"heat dissipation" 1
		"hull" 3
		"mass" 100
		"drag" 5
		"automaton" 1
		"thrust" 100
		"turn" 6000
	outfits
		"Cloak Checker"
	turret 0 0

outfit "Cloak Checker"

> 	category "Turret"

Reviewer: ```suggestion
	category "Turrets"
```

	thumbnail "outfit/unknown"
	"turret mounts" -1
	weapon
		"velocity" 10
		"lifetime" 1
		"reload" 1
		"blast radius" 20
		"hull damage" 5
		"no damage scaling"
		"range override" 1000000
		"turret turn" 360

mission "Ka'het: Patir Mystery 6"
	name "Second contact"
	description "Return to <system> with Dusk and four other xenoarchaeologists to continue your exploration of the <planet>. You will need to be cloaked as you enter to avoid alerting the Ka'sei."
	source
		government "Remnant"
		attributes "remnant primary"
	destination "Builder Settlement"
	passengers 5

> 	blocked `As soon as you arrive, Dusk contacts you again. "Welcome back <first>! We should be ready for the next Patir expedition, but we need at least 5 free bunks to attempt it."`

Reviewer: ```suggestion
	blocked `As soon as you arrive, Dusk contacts you again. "Welcome back, <first>! We should be ready for the next Patir expedition, but we need at least five free bunks to attempt it."`
```

	to offer
		random < 60
		has "event: patir mystery timer"
		not "ka'het: attacked ka'sei"
	on offer
		conversation
			`It doesn't take long for you to be noticed in the crowded, yet quiet, starport of <origin>. You feel a tap on your shoulder and turn around to see Dusk, accompanied by a group of four. "Good to see you here again, <first>! Do you remember our exploration of the Patir black hole?"`
			branch "atmospheric data"
				has "atmospheric scan intact"

> 			`	One of the other Remnant shows you a hologram of the system, with what looks like the Builder settlement you visited earlier highlighted by a yellow glow. "After studying the data we gathered during our last visit, we may have devised a way to enter the structure undisturbed! Based on our recordings, the Fetri'sei that attacked us in the structure was gathering materials from the asteroid belt with its fellow Ka'sei, and only moved inside the dome when they spotted us."`

Reviewer: ```suggestion
			`	One of the other Remnant shows you a hologram of the system, with what looks like the Builder settlement you visited earlier highlighted by a yellow glow. "After studying the data we gathered during our last visit, we may have devised a way to enter the structure undisturbed! Based on our recordings, the Fetri'sei that attacked us in the structure was gathering materials from the asteroid belt with its fellow Ka'sei, and only moved inside the dome when it spotted us."`
```

				goto proposal

			label "atmospheric data"
			action
				payment 300000
			`	One of the other Remnant shows you a hologram of the system, with what looks like the Builder settlement you visited earlier highlighted by a yellow light. "First of all, we have transferred <payment> to your logistics account for saving the data on the dome's atmosphere. It is only thanks to you that we were able to learn more about what kind of atmosphere the Builders may have lived in."`
			choice
				`	"Thank you."`
					goto skipdialog
				`	"Can you tell me more about that?"`
			`	"It seems their atmosphere may have been similar to ours - the air in the settlement we visited was almost evenly split between oxygen and argon: breathable by human standards."`

> 			`	"We could have removed our helmets after the airlock?" you ask.`

Author: I think it works best as a leading question in this context, but I'm open to changing it if necessary

> 			`	"It is a bit more complicated than that. We found that the pressure inside the complex was much, much lower than what we are used to - less than a sixteenth of an atmosphere. On most Earth-like planets, you would only find such pressures at over 20,000 meters. There are two possibilities: either the Builders were born in a thin atmosphere, or the life support system failed some time ago and the structure has been slowly losing air ever since. We hope to find the answer soon enough."`

Author: They would know how it is now, but it's less clear how it used to be when the Builders' race was born - it's been a very long time, and it is known that the Builders were capable of some terraformation (Eavine)

			label skipdialog

> 			`	"On top of that, we have learned that the Fetri'sei that attacked us was not waiting in the structure when we first entered the system; it was gathering materials from the asteroid belt together with the rest of its fellow Ka'sei, and only moved inside the dome when they spotted us."`

Reviewer: ```suggestion
			`	"On top of that, we have learned that the Fetri'sei that attacked us was not waiting in the structure when we first entered the system; it was gathering materials from the asteroid belt together with the rest of its fellow Ka'sei, and only moved inside the dome when it spotted us."`
```

			choice
				`	"I'm in."`
					accept
				`	"What if they detect us while inside the structure?"`
				`	"Sorry, but I don't think it is a good idea to return there."`
					decline
			`	"It is a possibility, yes. It is also quite problematic because communication between the inside and the outside of the structure is impossible. I think the best course of action would be to place a scanner that controls the large airlocks on the upper side of the dome, the same ones the Ka'sei used last time. If I am right, it should give us around a quarter of an hour to get back in the airlock and escape."`
			choice
				`	"Alright, let's do it."`
					accept
				`	"Sorry, but I don't think it is a good idea to return there."`
					decline

	on accept
		payment 100000
		set "remnant: dusk busy"
	on abort
		# Do nothing.
	on fail
		dialog

> 			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`

Author: Ideas on how to work around this? The easiest way would be to give up on using an in-universe explaination and use a generic "x or y happened", I guess

> 			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`

Author: Changed to `You have approached the Builder Settlement uncloaked, making the Ka'sei follow you, or the escort that was carrying the Remnant scientists has been destroyed. You have failed this mission.`

> 			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`

Reviewer 2: > Also, why are you approaching the planet?

...Because the settlement is on it? Also, this suggestion changes "As soon as you approach the Builder Settlement," to "As soon as you approach Builder Settlement".

> 			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`

Author: In my opinion, arriving in the system for the mission can be considered "approaching the settlement" even if you haven't explicitly started the landing itself, so it should work either way. I think "the builder settlement" flows better here, but i don't mind either way

	npc save
		government "Test Dummy"

> 		personality unconstrained heroic nemesis

Reviewer: ```suggestion
		personality heroic nemesis unconstrained
```

			`	"Great work, Captain. We are ready to start our mission." Dusk and his team are carrying with them a set of instruments similar to the first time you were here, from compact laser scanners to a pair of large crowbars. On the crew's backs are cylinders of varying dimensions, with the largest being almost a meter wide. You follow them through the ship's airlock onto the surface and begin the walk towards the complex. Halfway to the dome, Dusk speaks over the radio. "<first>, before we arrive there, do have any further questions on the mission plan?"`
			choice
				`	"What should I do once we are inside?"`
					goto inside
				`	"What are you and your team going to do in the complex?"`
					goto dusk
				`	"Do you think the Ka'sei are going to leave us alone the entire time?"`
					goto kasei
				`	"Don't worry, I'll figure it out."`
			`	"Well, I really hope so!" Dusk replies, grinning behind the glass of his helmet.`
				goto entrance

			label inside

> 			`	"Well, that is up to you. From the brief scans I made of the structure it does not seem to have particularly thick internal walls, so communication between us within the dome should be feasible. You could come with me inside the structure to help identify what we find, or if you prefer you could also remain near the airlocks and keep an eye for incoming Fetri'sei. As I said, the choice is yours."`

Reviewer: ```suggestion
			`	"Well, that is up to you. From the brief scans I made of the structure it does not seem to have particularly thick internal walls, so communication between us within the dome should be feasible. You could come with me inside the structure to help identify what we find, or if you prefer you could also remain near the airlocks and keep an eye out for incoming Fetri'sei. As I said, the choice is yours."`
```

				goto entrance

			label kasei
			`	"Honestly? At this point, I am positive they will. The last time we came here, they had already found us and were probably starting to enter the dome. Right now, they are still in the asteroid belt continuing their mission. There's the possibility that the airlock will alert them of our arrival, but at this point it is unlikely. In either case, I will be sure to keep an eye on the airlock once we are inside the dome."`
			label entrance
			`	It doesn't take long to reach the airlock ring in front of the dome. The airlock you used last time is still open, and its control panel is visible next to it. Dusk, following your gaze, quickly stops you. "I doubt it would be a good idea to pass through the same airlock we used last time, <first>," he says. "It looks like the structure's automated systems patched up the hole left by the Ka'sei, but I would never trust it as a pressure vessel at this point. Plus, I count at least six more airlocks just here."`

> 			`	He and a member of the group move to the opposite side of the control station next to the airlock, then flip one of the switches on the panel. Just like last time, a door opens next to them, revealing another on the opposite side of the small room. There is space only for three crew members at a time; you, Dusk, and another Remnant enter the airlock, while the other three wait their turn. The door closes, and Dusk flips the switch to begin the pressurization procedure. Suddenly, the lights inside the module turn on, emitting a bright, azure light.`

Reviewer: ```suggestion
			`	He and another member of the group move to the opposite side of the control station next to the airlock, then flip one of the switches on the panel. Just like last time, a door opens next to them, revealing another on the opposite side of the small room. There is space only for three crew members at a time; you, Dusk, and another Remnant enter the airlock, while the other three wait their turn. The door closes, and Dusk flips the switch to begin the pressurization procedure. Suddenly, the lights inside the module turn on, emitting a bright, azure light.`
```

			`	Some time later the group returns, satisfied with what they've found inside. "It was an interesting trip for sure, Captain," Dusk tells you. "Did anything of note happen out here?"`
			choice
				`	(Tell him about the light.)`
					goto explain
				`	"Not much. It was a bit boring."`
			`	"I imagine so. You would have been more helpful in the complex together with me." After picking up the scanners, you begin to pass through the airlocks, three people at a time. After a few minutes, you're all back outside the dome, ready to finally return to <planet>.`
				goto ending

			label alarm
			`	As you tell him about the blinking light, he interrupts you and announces to the rest of the group that they are leaving. "The yellow light is an indicator that the battery is running out," he explains hurriedly. "I should have checked our equipment better. Soon, the starship airlock will be unwatched." Instead of going through a single airlock, the group splits, half going into one airlock and half into another. When you're finally all outside of the dome, the tense mood dissipates.`
				goto ending

			label explain

> 			`	As you tell him about the blinking light, his expression changes. "You should have called me! We could have died!" He moves to the console on the laser scanner, which stays dark. "The yellow light was an indicator that the battery was running out," he says. "I should have checked our equipment better. The starship airlock may have remained unwatched for hours now. We need to leave." Instead of going through a single airlock, the group splits, half going into one airlock and half into another. When you're finally all outside of the dome, the tense mood dissipates..`

Reviewer: ```suggestion
			`	As you tell him about the blinking light, his expression changes. "You should have called me! We could have died!" He moves to the console on the laser scanner, which stays dark. "The yellow light was an indicator that the battery was running out," he says. "I should have checked our equipment better. The starship airlock may have remained unwatched for hours now. We need to leave." Instead of going through a single airlock, the group splits, half going into one airlock and half into another. When you're finally all outside of the dome, the tense mood dissipates.`
```

			choice
				`	"What do you think happened here?"`
					goto theory
				`	"Maybe the rest of the depot will have something."`
					goto samples
				`	"Is it possible that they just emptied the storage when they left?"`
			`	"Maybe. But look at how pristine this place is. There is no dirt or dust anywhere I can see, and everything seems to be exactly where it should. It seems more as if it was never inhabited in first place. There are far too many unknowns about Patir and this settlement to be certain of anything, though." He pulls another instrument from a waist pouch, then begins gathering samples from around the depot, both from the ground and on the shelves. The procedure takes a few minutes; he then puts them in one of the cylinders on his back and starts moving to the next room, signing to follow him.`
				goto depot

			label theory

> 			`	"At first I thought they abandoned this place and took everything with them, but look: There is no dirt or dust anywhere I can see, and everything seems to be exactly where it should. It seems more as if it was never inhabited in first place. There are far too many unknowns about Patir and this settlement to be certain of anything, though." He pulls another instrument from a waist pouch, then begins gathering samples from around the depot, both from the ground and on the shelves. The procedure takes a few minutes; he then puts them in one of the cylinders on his back and starts moving to the next room, signing to follow him.`

Reviewer: ```suggestion
			`	"At first I thought they abandoned this place and took everything with them, but look: there is no dirt or dust anywhere I can see, and everything seems to be exactly where it should. It seems more as if it was never inhabited in first place. There are far too many unknowns about Patir and this settlement to be certain of anything, though." He pulls another instrument from a waist pouch, then begins gathering samples from around the depot, both from the ground and on the shelves. The procedure takes a few minutes; he then puts them in one of the cylinders on his back and starts moving to the next room, signing to follow him.`
```

				goto depot

			label samples
			`	"Maybe, but I would not get my hopes up if I were you." Sitting down, Dusk takes out of his waist pouch a slim, organic-looking instrument, connected with a cable to a larger box still inside the bag. He takes samples from everywhere in the room: the shelves, the door, the walls, even some from the floor. After enclosing everything inside cylindrical containers, he tells you to get ready to move to another compartment.`
			label depot
			`	You follow Dusk around the building, but the results are the same. The depot is full of empty, untouched containers and barren racks in an incredible state of preservation. After Dusk gathers the last samples, you follow him outside, where you join the other teams leaving the building with their samples. Next stop: <planet>.`
				accept
			label gardens
				action

> 					"patir: builder seeds" ++

Reviewer: Do these two lines need to have one tab removed from them?

				accept
	on complete
		payment 50000
		conversation
			`After disembarking from the <ship>, you help the Remnant carry out what they found in the Builder settlement. The other teams gathered a lot of artifacts during the trip, from oddly shaped vials to tall pieces of alien machinery; you help Dusk carry a small crate, containing a dozen or so cylindrical objects roughly as large as your hand behind a transparent lid. "Thank you for your help, Captain."`
			choice
				`	"What is the next move?"`
					goto next
				`	"What is in that crate?"`
			`	"Oh, those? They are a Builder sidearm - a weapon capable of emitting short range electrical bursts strong enough to incapacitate or kill your target. We discovered a few hundred of them inside some bunkers in the Graveyard long ago. If you would like, I would not mind misplacing one if you are interested in them, as a token of thanks for all your help here."`
			scene "outfit/ka'het taser"
			choice
				`	"Thank you. How do I use it?"`
					goto outfit
				`	"I'd rather be paid in credits instead."`

> 			`"Oh, I have already transferred 50,000 credits to your account. I need to organize some things for our next mission; if you meet me in the spaceport a few hours from now, I should have everything ready.`

Reviewer: ```suggestion
			`	"Oh, I have already transferred 50,000 credits to your account. I need to organize some things for our next mission; if you meet me in the spaceport a few hours from now, I should have everything ready."`
```

				accept

			label outfit
			action
				outfit "Ka'het Electric Stunner" 1
			`	Dusk quickly explains how to use the Ka'het weapon; its short cylindrical shape is evidently not made for human hands and proves to be quite hard to use, but you eventually figure out how to hold it while still being able to readily press the trigger. While they may not be the best weapon to capture an enemy ship, they're surely going to be very handy from a defensive position inside your own.`
			label next
			`	"Now that we know how to safely access the site, I am going to need to organize our next mission. If you meet me in the spaceport a few hours from now I should have everything ready."`

mission "Ka'het: Patir Mystery 8A"
	name "Lab deployment"
	description "After taking the ground lab (weighing <tons>) from <stopovers>, reach <system> while still cloaked in order to set it up."
	source "Aventine"
	stopover "Babiali"
	destination "Builder Settlement"
	passengers 5
	cargo "research lab and satellite" 97

> 	blocked `Dusk contacts you soon after you reach the spaceport. "<first>, I advise you that the next mission is going to require <capacity>.`

Reviewer: ```suggestion
	blocked `Dusk contacts you soon after you reach the spaceport. "<first>, I advise you that the next mission is going to require <capacity>."`
```

			`	"Because of that, before we can start our mission, you are going to need to make a stopover at <stopovers> - I was able to convince the prefects on Viminal to reallocate one of the outposts that they had just delivered there to our mission, given it has higher priority status compared to the mining expeditions they are setting up."`
			`	You help Dusk transport the chosen instruments to your ship; you also make sure to leave enough space for the facilities you're going to take from <stopovers>. As you're walking back toward the ship's controls, Dusk reminds you to bring as many ramscoops with you as possible - you're going to need them in order to complete the mission plan while cloaked.`
				accept
	on stopover

> 		dialog

Author: Why?

> 		dialog

Reviewer: It's quite long for a dialog  (yes, I know they can scroll now).
Also, I'm not sure text replacements like `<stopovers>` always work well in dialogs.

			`	"Maybe, but that might be an unnecessary risk. First things first, <first>, there is no need to rush anything."`
				accept

	npc assist save
		system "Patir"
		government "Remnant"
		personality derelict target pacifist mute
		ship "Remnant Satellite" "Bright Ember Yonder"
		dialog `From the open cargo bay, you watch as the satellite moves up inside your ship. You split up tasks between yourself and Dusk, keeping the satellite stabilized while he downloads the data. In a few minutes the process is over, and you can return the probe to its original mission.`
		
	on complete
		conversation
			`Walking down your ship's stairway, Dusk thanks you again for your help. You hand him a datapad with the info gathered, as well as the equipment he brought with him to pick up the satellite's signal.`
			`	"I will join the rest of the team to start analysis," he signs in response. "We have a lot of data to examine, and not much time. Meet us in the lab in a few hours."`

ship "Remnant Satellite"
	sprite "ship/remnant satellite"
	attributes
		category "Drone"
		hull 130
		mass 10
		drag 2
		"heat dissipation" 1
	explode "tiny explosion" 5

> 	explode "small explosion" 2

Author: No, the satellite doesn't move.

				goto lab
			label plan
			`	"We have been considering a close approach to one of those beings ever since we first saw them, and your discovery of the planetoid in Lathia opens up the possibility for a surface expedition. Due to its size, landing with the <ship> should be possible, and we could even explore the planetoid in our suits if the surface is safe. Perhaps, if these beings are truly intelligent, this could be the first contact between our two species.`
			label lab
			`	"Meet us in the lab in a few hours, and we will have everything ready for the trip. With a bit of luck, most of the instruments we used on Chanai will be a good fit for this mission as well."`
		
mission "Ka'het: Patir Mystery 12B"
	name "Satellite deployment"
	description "Deploy two probes in <stopovers> to keep an eye on incoming space rocks."
	source "Builder Settlement"
	passengers 1
	cargo "Remnant Probes" 3
	waypoint "Mitera"
	waypoint "Thepa"
	to offer
		has "Ka'het: Patir Mystery 11: done"

> 		not "Ka'het: Patir Mystery 12A: accepted"

Reviewer 3: This is not a condition that will be automatically set and it doesn't look like you are setting it manually.
I'm guessing you want this mission to offer immediately after declining mission 12A, and not to offer at all if mission 12A is accepted?
In order to achieve that, I suggest something like:
```
mission "whatever 12B"
	to accept
		has "whatever 12A: offered"
		has "whatever 12A: declined"
```

Author: Used warp's suggestion

	on enter "Thepa"
		dialog
			`Dusk quickly reaches for the console, initiating the satellite deployment procedure you've become accustomed to. It doesn't take long for it to complete, and in a few minutes you're ready to leave the system once again.`

ship "Asteroid Blocker"
	attributes
		"heat dissipation" 1
		"hull" 3
		"mass" 100
		"drag" 5
		"automaton" 1
		"thrust" 100
		"turn" 6000
	outfits
		"Fuel Removal"
	turret 0 0

outfit "Fuel Removal"
	category "Turret"
	thumbnail "outfit/unknown"
	"turret mounts" -1
	weapon
		"velocity" 1000
		"lifetime" 10
		"reload" 100000
		"blast radius" 200000
		"safe"
		"no damage scaling"
		"range override" 10000000
		"homing" 4
		"tracking" 1
		"acceleration" 1.5
		"drag" 0.2
		"turn" 12
		"fuel damage" 500
		"turret turn" 360

mission "Ka'het: Patir Mystery 12C"
	landing
	name "Ka'sei hacking"
	description "Taking advantage of the arrival of one of the asteroids, berth with a downed Ka'sei to gather data from its internal computers."
	passengers 5
	source "Builder Settlement"
	to offer
		has "Ka'het: Patir Mystery 12B: done"
	on offer

> 		event "no Patir spaceport and no Fetri spawn"

Reviewer: ```suggestion
		event "no Patir spaceport and no Fetri spawn" 0
```

		conversation
			`Dusk seems busier than usual as you descend towards the settlement, constantly communicating with the team on the ground while simultaneously checking the incoming telemetry. When you're hovering right above the landing pad, he turns to your console. "Keep the thrusters in standby, Captain," he signs hastily. "We are not here to stay. The satellites are working very well; in fact, the Thepa probe has just detected one of those beings jumping right in our direction. We have only a few hours to return to orbit, or we will waste this opportunity."`

> 			`	On the ground, the whole team is outside the lab, their hands full of strange equipment and unwieldy machinery. You deploy your ship's ramp to deploy and quickly walk down with Dusk to help them carry the instruments aboard. "The approaching asteroid is of the small or medium kind, as far as I can tell. Considering your past interactions, or lack thereof, it is unlikely to cause any issue."`

Reviewer: Woa, this is adding a lot of lore...
```suggestion
			`	On the ground, the whole team is outside the lab, their hands full of strange equipment and unwieldy machinery. You set your ship's ramp to deploy and quickly walk down to Dusk so you can help them carry the instruments aboard. "The approaching asteroid is of the small or medium kind, as far as I can tell. Considering your past interactions, or lack thereof, it is unlikely to cause any issue."`
```

			`	"Of course, this will be the first time in Remnant history boarding a Ka'sei; after all, it wasn't too long ago when we learned of their existence. But we have explored many kinds of Ka'het over the years with varying results, and with that experience I am confident we should be able to complete our mission."`
			choice
				`	"Are you sure this will be safe?"`
				`	"Let's go then."`
					goto depart
			`	"Trust me, <first>," Dusk replies, "it will be incredibly risky. None of us would be doing this if we could say for certain there was a better way, but here we are. Help me with this crate now, we are almost ready to go."`
			label depart
			`	As the last container is brought inside, the cargo ramp moves back, and the exit door follows suit. Glancing at the scanner telemetry, you realize the alien being has already started to exit hyperspace; with a button press, the engines roar back to life, and the ship starts gaining altitude.`
				launch
	on enter

> 		event "Patir spaceport returns and Fetri spawn"

Reviewer: ```suggestion
		event "Patir spaceport returns and Fetri spawn" 1
```

	npc assist
		government "Ka'sei (12A)"

> 		personality derelict uninterested waiting plunders mining harvests

Reviewer: ```suggestion
		personality derelict harvests mining plunders uninterested waiting
```

			label fired
			`	Second after second passes relentlessly, with no reaction whatsoever from the Fetri'sei or its surroundings. As the unknown creature fades behind a violet cloud and jumps away, you can't help but wonder what to do.`
			choice
				`	(Shoot the Fetri'sei a second time.)`
					goto shoot
				`	(Keep waiting.)`
					goto wait
				`	(Jump on the Ka'sei to call the Remnant aboard.)`
			`	There's no time to waste. Without hesitation, you push yourself off the border of the airlock and toward the Fetri'sei. After remaining airborne for a few instants, you land right next to the leftmost engine nacelle, bouncing off its skin. A timely activation of the thrusters on your wrists helps you move back to its surface, however, and by grabbing a long spike attached to its back, you can finally stabilize and move further ahead.`
			`	White noise starts coming from your communicator; moving your head up, you finally see the Remnant leaping out of the hatch, carrying a dozen faintly glowing cylindrical tubes under their arms. They notice you right when communication returns to normal, and their message arrives loud and clear: "Jump!"`

> 			`	You leave the handles and use your feet to give you a good push away from the spacecraft. At the last second, you move to the right to avoid ending up behind one of its thrusters right as they roar to life. The Fetri'sei has awakened, and for an instant you see all of its systems activate at once - the engines, the steering, even the Ravagers.`

Reviewer: ```suggestion
			`	You leave the handles and use your feet to give you a good push away from the spacecraft. At the last second, you move to the right to avoid ending up behind one of its thrusters just as they roar to life. The Fetri'sei has awakened, and for an instant you see all of its systems activate at once - the engines, the steering, even the Ravagers.`
```

			`	One at a time, they move back into the airlock, leaving their cargo attached to one of the instruments they carried with them. As you close the door and pressurize the compartment, Dusk approaches you about the mission.`
			`	"Thank you for that, <first>. We had to cut short the exploration when we saw your message, before we had the chance to retrieve the specific data we were looking for. As a compromise, we decided to take some memory banks - these cylinders - with us, so we could search through the data later. It was dangerous, but it worked.`
			`	"That said, I suggest returning to the outpost as soon as possible. The Ka'sei we have boarded may start acting erratically after this, or it may even attack us; we have removed part of its brain, after all. Hopefully this mission will not backfire against us."`
			`	You sit back in the cockpit, starting the engines once again.`
	on complete
		fail "Ka'het: Patir Mystery 12D"

# The mission here is a complex mashup of game functions that uses many different mechanics to achieve the intended sequence:
# - two ships spawn, one disabled and one functional (but otherwise standing still);

> # - the player boards the disabled ship, with the mission happening before his eyes;

Reviewer 3: Not in my undefined gender protagonist game.
```suggestion
# - the player boards the disabled ship, with the mission happening before their eyes;
```

# - the other ships jump away right after the mission, and the one the player boarded returns to its normal functions.

> # In order to achieve this, the asteroid ship has just over 200 fuel and a negative fuel generation, meaning that it will be unable to jump essentially right after it spawns; removing the spaceport means that it won't be able to refuel on the Builder settlement, and so it remains in space doing nothing but waiting. When the player boards the disabled NPC, it is repaired and returns to its functions; as it is an enemy of the asteroid ship, it will shoot it with every available weapon, including what has been hidden under "Ka'het Machinery": a negative fuel damage weapon, that will instantly refill its fuel reserves and let it jump. Giving it just a capacity of 200 would mean that the game would always think it is just infinitesimally under the threshold to jump, so I gave it 250 and everything works as it should with surprising consistency. Two peculiarities of this are that the player will be able to board the asteroid ship to refuel it, and that the Ka'sei could shoot the asteroid with its ravagers for a split second before it jumps if the position of the two is just right. Both of them are harmless: the health of the asteroid is orders of magnitude higher than the damage the ka'sei can inflict in that time, and as the player boarding the asteroid will give it exactly 200 fuel because of game mechanics, 1 single frame later it will be already unable to jump. However, in order to account for their oddity I added a small dialog to the asteroid's boarding screen to make it as if you were flying by it, and described at the end of the previous mission how for an instant the Ravagers ignite when the Ka'sei returns online, as if in an unintended burst.

Reviewer: Wow.
```suggestion
# In order to achieve this, the asteroid ship has just over 200 fuel and a negative fuel generation, meaning that it will be unable to jump essentially right after it spawns; removing the spaceport means that it won't be able to refuel on the Builder settlement, and so it remains in space doing nothing but waiting. When the player boards the disabled NPC, it is repaired and returns to its functions; as it is an enemy of the asteroid ship, it will shoot it with every available weapon, including what has been hidden under "Ka'het Machinery": a negative fuel damage weapon, that will instantly refill its fuel reserves and let it jump. Giving it just a capacity of 200 would mean that the game would always think it is just infinitesimally under the threshold to jump, so I gave it 250 and everything works as it should with surprising consistency. Two peculiarities of this are that the player will be able to board the asteroid ship to refuel it, and that the Ka'sei could shoot the asteroid with its ravagers for a split second before it jumps if the position of the two is just right. Both of them are harmless: the health of the asteroid is orders of magnitude higher than the damage the Ka'sei can inflict in that time, and as the player boarding the asteroid will give it exactly 200 fuel because of game mechanics, 1 single frame later it will be already unable to jump. However, in order to account for their oddity I added a small dialog to the asteroid's boarding screen to make it as if you were flying by it, and described at the end of the previous mission how for an instant the Ravagers ignite when the Ka'sei returns online, as if in an unintended burst.
```

event "no Patir spaceport and no Fetri spawn"
	system Patir
		remove fleet "Ka'sei"
	planet "Builder Settlement"
		remove spaceport

event "Patir spaceport returns and Fetri spawn"
	system Patir
		add fleet "Ka'sei" 300
	planet "Builder Settlement"
		spaceport `The Remnant laboratory merges oddly well with the rest of the environment, similar in color and design to the Builder structures around here. The spaceport amounts only of a flat strip of regolith near the settlement, hastily cleaned of the debris that covers most of the terrain.`

mission "Ka'het: Patir Mystery 12D"
	landing
	invisible
	source "Builder Settlement"
	to offer
		has "Ka'het: Patir Mystery 12B: done"
	npc assist
		government "Magic Asteroid"

> 		personality coward waiting pacifist harvests uninterested fleeing

Reviewer: ```suggestion
		personality coward fleeing harvests pacifist uninterested waiting
```

		fleet

> 			names "ka'sei"

Reviewer: ```suggestion
			names "Magic Asteroid"
```

			cargo 3
			variant
				"Asteroid Large (Patir 12A)"
		dialog

> 			`The giant creature stands before you, slowly pulsating in a pale pink halo. The bubbles that interrupt the asteroid-like surface are slightly semitransparent, giving you a brief look at the crystalline structure inside; but your mind soon returns to your time limited mission, and you decide to move ahead before it's too late.`

Reviewer: ```suggestion
			`The giant creature stands before you, slowly pulsating in a pale pink halo. The bubbles that interrupt the asteroid-like surface are slightly semitransparent, giving you a brief look at the crystalline structure inside; but your mind soon returns to your time limited mission, and you decide to move ahead before it's too late.`
	to fail
		has "Ka'het: Patir Mystery 12C: done"
```

			`	When the scientists finish unloading their cargo, Dusk calls you with the communicator. Now that the immediate danger is over, a hint of frustration creeps into his voice.`
			`	"The cells we have taken from the Fetri are only a few out of multiple hundreds. We did our best to choose those that were recorded at times when those beings were present, but we will still need some luck to get the results we wanted.`
			`	"Our calculators have been crunching numbers for hours now, ever since we cracked open the first canister. By the time you have reached the lab, we should have the analysis ready." You close the hatch behind you to join the rest of the crew.`
			`	As the asteroid continues its nearly imperceptible spin, Patir emerges from behind the giant dome, allowing its rays to pass through. You take the last steps towards the outpost with the sunshade lowered and walk into the lab once more.`

> 			`	The module was always cramped, but over the last few weeks, a thin layer of dust coming from outside the asteroid has covered the floor, with just the surfaces needed for their research kept clean. Tiredness plagues the faces of the scientists: they must not have slept much in the past days to keep up with the amount of incoming data, and neither have you. As you try to remember the last time you stepped outside without a suit - at least a month, most likely - your stream of thoughts is interrupted by the arrival of Dusk, carrying in his hands a ream of papers.`

Reviewer: This sentence is a duplicate of another mission's. Perhaps they should be changed to not be exactly the same?

Author: The two paths are separate

			`	"The information in the memory containers is indeed from when when one of those beings was in the system, and the data recorded was far more comprehensive than our own scans. However, the memory containers we extracted only contain records from the middle of the phenomenon, without showing how it begun or ended. Additionally, understanding their contents is going to take longer than we hoped for. We knew that the exact relationship between the being and the Fetri'sei would be beyond our knowledge, but we cannot even make preliminary guesses as to how they are able to communicate.`
			`	"We have enough consumables to last for many weeks still; we started rationing as soon as we arrived in this sector, and we are prepared to reduce them further if necessary. Despite this, the time and manpower needed to crack this code would go far beyond what we could feasibly achieve. We have hit a roadblock that, for now, we cannot overcome."`
			choice
				`	"Does this mean we need to repeat the mission?"`
				`	"What are our options?"`
					goto options

> 			`	"We considered it briefly, but it would be quite ineffective. Our blocker is understanding the data itself, and having even more would do little to help us with that. On top of that, the risk would be extremely high, as we have seen just earlier. All things considered, we are at the same point we were a week ago.`

Reviewer: ```suggestion
			`	"We considered it briefly, but it would be quite ineffective. Our blocker is understanding the data itself, and having even more would do little to help us with that. On top of that, the risk would be extremely high, as we have seen. All things considered, we are at the same point we were a week ago.`
```

			label options
			`	"We are left with few alternatives. As I told you previously, the Chanai station is an interesting possibility, and could have influenced Patir's erratic behavior. At the same time, exploring the asteroid around Lathia may put us closer to understanding the inner workings of the beings and how they interact with the Ka'sei. It is quite dangerous, but visiting the space station is also an unknown, and maybe a more direct approach is what we need to accomplish our objective. Neither mission prevails over the other in a definite way, and our votes within the team are split evenly between the two, so if y-"`
			`	Dusk halts mid phrase, looking behind you with an awestruck gaze. The sky has again turned black, and a few dim lines are already starting to appear in the distance. The whole team jumps to the windows, watching as the light show begins.`

> 			`	While stars and galaxies zoom in the night sky, a few of the Remnant run around the lab, turning on various instruments. However, most of them remain with their faces stuck to the glass. The lights start converging toward a single point, forming a large, but rapidly shrinking mass in the background.`

Reviewer: > a few of the Remnant

Well, if few means three or four (at a conservative estimate) and there are only five Remnant...

	"reputation: Ka'sei" = 1

event "patir mystery timer"

ship "Cloak Check"
	attributes
		"heat dissipation" 1
		"hull" 3
		"mass" 100
		"drag" 5
		"automaton" 1
		"thrust" 100
		"turn" 6000
	outfits
		"Cloak Checker"
	turret 0 0

outfit "Cloak Checker"
	category "Turret"
	thumbnail "outfit/unknown"
	"turret mounts" -1
	weapon
		"velocity" 10
		"lifetime" 1
		"reload" 1
		"blast radius" 20
		"hull damage" 5
		"no damage scaling"
		"range override" 1000000
		"turret turn" 360

mission "Ka'het: Patir Mystery 6"
	name "First contact"
	description "After a tumultuous first contact, return to <destination> without harming any Ka'sei, even if they attack you."
	source

> 		government "Remnant"

Author: Changed

	"reputation: Ka'sei" = 1

event "patir mystery timer"

ship "Cloak Check"
	attributes
		"heat dissipation" 1
		"hull" 3
		"mass" 100
		"drag" 5
		"automaton" 1
		"thrust" 100
		"turn" 6000
	outfits
		"Cloak Checker"
	turret 0 0

outfit "Cloak Checker"
	category "Turret"
	thumbnail "outfit/unknown"
	"turret mounts" -1
	weapon
		"velocity" 10
		"lifetime" 1
		"reload" 1
		"blast radius" 20
		"hull damage" 5
		"no damage scaling"
		"range override" 1000000
		"turret turn" 360

mission "Ka'het: Patir Mystery 6"
	name "First contact"
	description "After a tumultuous first contact, return to <destination> without harming any Ka'sei, even if they attack you."
	source
		government "Remnant"
	destination "Builder Settlement"

> 	passengers 5

Author: Added

			`	You help Dusk transport the chosen instruments to your ship; you also make sure to leave enough space for the facilities you're going to take from <stopovers>. As you're walking back toward the ship's controls, Dusk reminds you to bring as many ramscoops with you as possible - you're going to need them in order to complete the mission plan while cloaked.`
				accept
	on stopover
		dialog
			`Dusk's coordinates indicate the border of a large canyon near the equator of <planet>. Unlike most cracks on the surface, this one shows signs of geological activity: you only narrowly avoid a stream ejecting from a geyser in the canyon as you pass overhead. As you land, a row of Remnant structures are clearly visible running parallel to the cliff.`
			`	The local crew assist in loading the modular laboratory split across three pallets; with them arrives a separate package, containing a surveillance satellite to be deployed near Patir to scan the Ka'sei from orbit.`

> 	on fail

Author: Clever

	on enter Athiri
		dialog
			`As you enter the system, your sensors lose any trace of the wormhole you passed from. There doesn't seem to be a way to pass through it again from this end.`
	on visit
		dialog
			`Observing the systems you have mapped, Dusk seems disappointed. "There must be a way to reach the systems in the north. Please look everywhere you can for a passage; there has to be something."`

event "Patir moves"
	system Patir
		pos 10059.5 10.5
	planet "Builder Settlement"
		spaceport `The Remnant laboratory merges oddly well with the rest of the environment, similar in color and design to the Builder structures around here. The spaceport amounts only of a flat strip of regolith near the settlement, hastily cleaned of the debris that covers most of the terrain.`

mission "Ka'het: Patir Mystery 10"
	landing
	name "Gather data"
	description "Board the Remnant satellite in orbit around Patir to learn more about the odd creatures around here."
	passengers 1
	source "Builder Settlement"
	to offer
		has "Ka'het: Patir Mystery 9: done"
	on offer
		conversation

> 			`Your ship's antennae started transmitting the information gathered to Dusk as soon as you jumped here; the entire data package has been delivered before you've even reached the Builder settlement. Wasting no time, the Remnant have analyzed the majority of the data by the time you enter the lab, and seem to be discussing some of the results. When they notice you, they abandon their current conversation and move closer to learn more about what you've found.`

Author: I'm copypasting all these suggestions manually since github isn't working right; changed "hear from" to "speak with" as per ziproot's comment

		
	on complete
		conversation
			`Walking down your ship's stairway, Dusk thanks you again for your help. You hand him a datapad with the info gathered, as well as the equipment he brought with him to pick up the satellite's signal.`
			`	"We are going to need some time to analyze what we have found now, Captain," he signs in response. "I will meet up with the rest of the team to start the process. We have a lot of data to examine, and not much time."`
			choice
				`	"How long will it take?"`
			`	"It is hard to say exactly, but it will be a few hours at least. Meet us in the lab after that, and we should be ready to tell you the next step."`

ship "Remnant Satellite"
	sprite "ship/remnant satellite"
	attributes
		category "Drone"
		hull 130
		mass 10
		drag 2
		"heat dissipation" 1
	explode "tiny explosion" 5
	explode "small explosion" 2

mission "Ka'het: Patir Mystery 11"
	name "Gather more data"

> 	description "Visit the neutron star to study more thoroughly the mysterious creatures."

Author: No reason, so I added it to this one and fixed a few other instances of writing down the system name instead of the keyword

			`	As he's speaking, one of the other Remnant shows you a hologram of Patir and its system, with what looks like the Builder settlement you visited earlier highlighted by a yellow glow. "After studying the data we gathered during our last visit, we may have devised a way to enter the structure undisturbed! Based on our recordings, the Fetri'sei that attacked us was not waiting in the structure when we first entered the system; it was gathering materials from the asteroid belt together with the rest of its fellow Ka'sei initially, but changed path and moved inside the dome to await us as soon as we were noticed. If we could enter the system without being noticed, we could perhaps be able to explore the structure freely!`
				goto proposal

			label "atmospheric data"
			action
				payment 300000

> 			`After disembarking from the <ship>, you find yourself once again in the crowded, but quiet starport of <origin>. It doesn't take long for you to be noticed, however: only a few minutes later, you feel a tap on your shoulder and turn around to see Dusk, accompanied by a group of five people dressed in Remnant attire. "Good to see you here again, <first>! I have good news. Do you remember our exploration of the Patir black hole?"`

Author: Removed the repetition

			choice
				`	"I'm in."`
					goto in
				`	"What if they detect us while inside the structure?"`
				`	"Sorry, but I don't think it's a good idea to return there."`
					decline
			`	"It is a possibility, yes. It is also quite problematic because once we are in the airlock, it will be impossible to communicate through the structure to the outside, and vice versa. I think the best course of action would be, as soon as we are in, to place a scanner that controls the large airlocks on the upper side of the dome, the same ones the Ka'sei used last time; if I am right, it should give us around a quarter of an hour to get back in the airlock and escape safely."`
			choice
				`	"Alright, let's do it."`
				`	"Sorry, but I don't think it's a good idea to return there."`
					decline

			label in
			`	"Perfect. Remember to equip as many ramscoops as you can - they are going to be very useful this time."`

> 				accept

Author: It isn't meant to handhold the player, but to ensure to prepare them for what will happen after the lab deployment (unbeknownst to Dusk or first last) - the missions thereafter will be harder and more frustrating without a sizeable amount of fuel or a good ramscooping capability. I've just used the excuse of the cloaked mission to allow a character to be able to warn the player

> 				accept

Author: The most fuel efficent roundtrip to Patir from the furthest remnant world is only 1400 fuel, and after 700 you can ramscoop very very quickly from Patir itself; during the mission chain instead 2200 fuel is needed, and this right after already doing a trip to Patir. While a few spots on the way allow for some ramscooping, if possible I'd like to leave at least the first mention, removing the second one a few missions after this

> 				accept

Reviewer 4: If you want to add an incentive to install a ramscoop, then why not also add a condition where you have to stay in the system for a while, instead of just landing on the settlement?

> 				accept

Author: Eh, I don't think that flows very well, and I'm not a big fan of "stay in a system doing nothing for x time" mechanics. I'll just remove these, they're not strictly essential

	on accept
		payment 100000
		set "remnant: dusk busy"
	on fail
		dialog

> 			`The Ka'sei saw your ship and started following you, making exploration impossible. You have failed this mission.`

Author: There is no way to tell if the player cloaked at all, or if they had brought a cloak in first place - all is known is that they failed the mission. I'll reword it

> 			`The Ka'sei saw your ship and started following you, making exploration impossible. You have failed this mission.`

Author: Replaced with ```As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.```

	npc save
		government "Test Dummy"
		personality unconstrained heroic nemesis
		system "Patir"
		fleet
			variant
				"Cloak Check" 10

mission "Ka'het: Patir Mystery 7"
	landing

> 	name "Return to <system>"

Author: None that I can think of, changed

			choice
				`	"What should I do once we're inside?"`
					goto inside
				`	"What are you and your team going to do in the complex?"`
					goto dusk
				`	"Do you think the Ka'sei are going to leave us alone the entire time?"`
					goto kasei
				`	"Don't worry, I'll figure it out."`
			`	"Well, I really hope so!" Dusk replies, grinning behind the glass of his helmet.`
				goto entrance

			label inside
			`	"Well, that's up to you. From the brief scans I made of the structure it does not seem to have particularly thick internal walls, so communication between us within the dome should be feasible. You could come with me inside the structure to help identify what we find, or if you prefer you could also remain near the airlocks and keep an eye on the large one for incoming Fetri'sei. As I said, the choice is yours."`
				goto entrance

			label dusk

> 			`	"It depends. Here, look," he says, passing you a data pad showing a rather blurry picture. "It is a still image taken from the recording of our encounter with the Fetri'sei. The strong lights ruined most of the video, so we will to have to settle for this picture, but look at the background: do you see that curved board on the left? The quality of the video is too low to be sure, but I think it could be a map of the complex. If I am right, we could split into groups and cover more ground in the few hours we are going to spend here." "What if you aren't?" you ask. "Well, in that case we will just proceed with our mission one building at a time. But I really do think that is a map that we are seeing here; it would be the perfect place to put one."`

Author: Added the full suggestion manually to avoid commit clutter

				goto depot

			label samples
			`	"Maybe, but I would not get my hopes up if I were you. We will see." Sitting down on the ground, Dusk takes out of his waist pouch a slim, organic-looking instrument, connected with a cable to a larger box still inside the bag. He takes samples from everywhere in the room: the shelves, the door, the walls, even some from the floor. After enclosing everything inside cylindrical containers, he tells you to get ready to move to another compartment.`
			label depot
			`	You follow Dusk around the building, but the results are the same. The depot is full of empty, untouched containers and barren racks apparently in an incredible state of preservation for their age. After Dusk gathers the last samples, you follow him outside, where you join the other Remnant teams leaving the building with many samples to analyze. Next stop: <planet>.`
				accept
			label gardens
				action
					"patir: builder seeds" ++

> 			`	"I agree, it feels the most interesting of the two." The two of you follow the route indicated on the map for a while before reaching the entrance: a slender, sturdy-looking pillar roughly two meters wide with a circular opening on one side. Moving closer you can see the end of a shallow slide right beyond the door, swirling upwards inside of the column and likely reaching the top of the dome.`

Author: You're right, changed

			`	"If this was supposed to be the exit of the garden," Dusk says, "I fail to see where the entrance should be. We are going to have to walk it up from here instead."`
			choice
				`	"Alright, let's go."`
					goto slide
				`	"I would rather keep searching for the entrance."`
			`	You and Dusk keep walking around the structure for almost half an hour to no avail; with no other choices left but going up the slide, you give up and decide to follow it up the pillar. Going through it is simpler than it looked, as the surface of the floor is ribbed in a peculiar way so that gripping onto it becomes rather effortless. It still takes a while to reach the end: even so, though the garden is nowhere near the top of the dome it takes another half an hour to reach it.`
				goto top

			label slide

> 			`	You follow Dusk up along the slide; the inside of the pillar is just as dark as the rest, and the length of it alone makes the ascent quite tiresome. Going through it is simpler than it looked, as the surface of the floor is ribbed in a peculiar way so that gripping onto it becomes rather effortless. It still takes a while to reach the end: even so, though the garden is nowhere near the top of the dome it takes another half an hour to reach it.`

Author: The darkness is meant to refer to the previously established lack of lighting in the outside, not to the color of the pillar

			`"Oh, I have already transferred 50,000 credits to your account. I need to organize some things for our next mission, however; if you meet me in the spaceport a few hours from now I should have everything ready.`
				accept

			label outfit
			action
				outfit "Ka'het Electric Stunner" 1
			`	Dusk quickly explains how to use the Ka'het weapon; its short cylindrical shape is evidently not made for human hands and proves to be quite hard to use, but you eventually find out how to hold it while still being able to readily press the trigger. While they may not be the best weapon to capture an enemy ship, they're surely going to be very handy from a defensive position inside your own.`
			label next
			`	"Well, I am going to need to organize some things for our next mission now that we know how to safely access the site. If you meet me in the spaceport a few hours from now I should have everything ready."`

mission "Ka'het: Patir Mystery 8A"
	name "Lab deployment"
	description "After taking the ground lab from <stopovers>, reach <system> while still cloaked in order to set it up."
	source "Aventine"
	stopover "Babiali"
	destination "Builder Settlement"
	passengers 5

> 	cargo "research laboratory and tracker probe" 97

Author: Changed to "research lab and satellite"

	to offer
		has "Ka'het: Patir Mystery 7: done"
	on offer
		conversation
			`It doesn't take long for you to see Dusk; he's waiting for you under a canopy next to your landing pad, right at the edge of Aventine's starport. "We are almost ready to go," he says, "but I have run into a problem. My plan for our next mission was to have a repeat of the last expedition, but this time carrying a temporary laboratory to the Builder site; it seems like the best way to avoid the limitations of the previous expedition. I was able to gather most of the instruments I was looking for in the local scientific facilities, but none of the mobile laboratories I am authorized to requisition by default were in good enough condition to fly."`
			`	"Because of that, before we can start our mission, you are going to need to make a stopover at <stopovers> - I was able to convince the prefects on Viminal to reallocate one of the outposts that they had just delivered there to our mission, given it has higher priority status compared to the mining expeditions they are setting up. Is your ship ready?"`
			choice
				`	"It is. Let's go."`

> 			`	You help Dusk transport the chosen instruments to your ship; even if, on their own, they only occupy a small portion of the <ship>'s cargo bay, you also make sure to leave enough space for the facilities you're going to take from <stopovers>. As you're walking back toward the ship's controls, Dusk reminds you to bring as many ramscoops with you as possible - you're going to need them in order to complete the mission plan while cloaked.`

Author: Same as above - while disguised as suggestions on how to act in relation to the need to cloak, in reality it's to ensure the player is going to be prepared for what comes after Patir moves

				accept
	on stopover
		dialog
			`Dusk's coordinates indicate the border of a large canyon near the equator of <planet>. Unlike most cracks on the surface, this one shows signs of geological activity: you only narrowly avoid a stream ejecting from a geyser in the canyon as you pass overhead. As you land, a row of Remnant structures are clearly visible running parallel to the cliff.`
			`	The local crew assist in loading the modular laboratory split across three pallets; with them arrives a separate package, containing a surveillance satellite to be deployed near Patir to scan the Ka'sei from orbit.`
	on fail
		dialog

> 			`The Ka'sei saw your ship and started following you, making the exploration impossible. You failed the mission.`

Author: Replaced with ```As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.```

	on enter Patir
		dialog
			`The probe slowly moves away from your ship, opening its external panels and the instrumentation; as you watch it turn into a dim point of light, the Remnant team signals you it's functioning correctly, and you prepare to approach the asteroid.`

	npc save
		government "Test Dummy"
		personality unconstrained heroic nemesis
		system "Patir"
		fleet
			variant
				"Cloak Check" 10

mission "Ka'het: Patir Mystery 9"
	landing
	name "Explore the neighborhood"
	description "Visit every system around Patir you can reach, then return to Dusk in <system>."
	source "Builder Settlement"
	waypoint "Athiri"
	waypoint "Chanai"
	waypoint "Ghila"
	waypoint "Lathia"
	waypoint "Maithi"
	waypoint "Mitera"
	waypoint "Sarifa"
	waypoint "Thepa"
	to offer
		has "Ka'het: Patir Mystery 8A: done"
	on offer
		event "Patir moves"
		conversation

> 			`Approaching the settlement from northeast, you park the <ship> once again next to airlock array; it's almost starting to become a familiar sight by now. With the cargo ramp pointed toward the entrance and the post-flight checklist completed, the deployment of the lab can finally commence; the Remnant team disembarks quickly and starts working on the first pallet, while you remain in the cockpit keeping an eye on the Ka'sei activity in the area. The researchers work quickly and nearly without issues in the light gravity of the asteroid, besides the occasional lost screw, but the construction job still takes nearly the entire day.`

Author: The ship and lab are meant to be uncloaked on the terrain, as it would needlessly complicate the operations. After all, walking out of the ship would make them detectable anyways, and the cloak itself is only meant to trick the ka'sei into thinking nobody new is visiting the lab.

Added the suggestion separately btw, to avoid spamming the PR with commits

			choice
				`	"How is that possible?"`
				`	"Where are we then?"`
				`	"How do we get back?"`
				`	"Was that a hyperdrive jump then?"`
			`	"I have no idea, <first>. What I can tell you for now is that we need to get a clearer idea of what we have around before deciding the next step, and there is no way to get that from here; I have to ask you to try exploring as far as you can go in this new sector, and return once you have reached the end or hopefully found a way out. I promise I will have a new plan ready that by the time you're back."`
			`	As you prepare the ship for liftoff, your mind keeps wandering to Dusk's words, and to the events of tonight. Returning back home won't be easy this time, that's for sure.`
				accept
	on enter Chanai
		dialog

> 			`As soon as you jump in, your sensors detect a massive structure of artificial nature, apparently a station of some kind. Given its strange structure and its proximity to Patir, the Remnant will be definitely interested in it.`

Author: Chani is the Fremen girl, Chanai is the system

			`	After he finishes signing, he makes a few quick gestures toward the projector, enlarging the hologram of the Builder settlement and revealing what looks like a flight path. "You are going to land roughly where we did last time, carrying me and four other xenoarcheologists. If you accept, I should be able to allocate <payment> to help you buy the necessary equipment for this mission."`
			choice
				`	"I'm in."`
					accept
				`	"What if they detect us while inside the structure?"`
				`	"Sorry, but I don't think it's a good idea to return there."`
					decline
			`	"It is a possibility, yes. It is also quite problematic because communication between the inside and the outside of the structure is impossible. I think the best course of action would be to place a scanner that controls the large airlocks on the upper side of the dome, the same ones the Ka'sei used last time. If I am right, it should give us around a quarter of an hour to get back in the airlock and escape."`
			choice

> 				`	"Alright, let's do it."`

Author: Ditto

> 				`	"Alright, let's do it."`

Author: Makes sense to me

					accept
				`	"Sorry, but I don't think it's a good idea to return there."`
					decline

	on accept
		payment 100000
		set "remnant: dusk busy"

> 	on fail

Author: What are this and the above suggestion for?

		dialog
			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`

> 	npc save

Reviewer: ```suggestion
	npc evade save
```

			`	"Great work, Captain. We are ready to start our mission." Dusk and his team are carrying with them a set of instruments similar to the first time you were here, from compact laser scanners to a pair of large crowbars. On the crew's backs are cylinders of varying dimensions, with the largest being almost a meter wide. You follow them through the ship's airlock onto the surface and begin the walk towards the complex. Halfway to the dome, Dusk speaks over the radio. "<first>, before we arrive there, do have any further questions on the mission plan?"`
			choice
				`	"What should I do once we're inside?"`
					goto inside
				`	"What are you and your team going to do in the complex?"`
					goto dusk
				`	"Do you think the Ka'sei are going to leave us alone the entire time?"`
					goto kasei

> 				`	"Don't worry, I'll figure it out."`

Author: This one (and a few others) are explicitly happening over the radio, I'd rather keep them as is since they flow better

			choice
				`	"Did you do that?"`
				`	"What happened?"`
			`	"I am not sure. Wait." Moving to the opposite side of the airlock, he uses a crowbar to open another hidden control station, which slides out of the wall to reveal a rather large screen scrolling through some alien writing. After reading through it with some difficulty, Dusk breathes a sigh of relief and pulls it back to its original location. "I think it is nothing," he adds, "at least I hope so. From what I could gather, the lights being switched on were a normal, timed event in the commanded startup sequence - why they they did not turn on last time is beyond me, but nothing seems to be out of the ordinary. Perhaps they were just broken."`

> 			`	A few minutes later, pressurization finishes, and the interior door opens. You follow Dusk out of the airlock and find yourself in front of a large area, likely the same place the encounter with the Fetri'sei took place. "The moment of truth," exclaims Dusk, placing down a heavy scanner, similar to a short telescope. "This is a low-power laser system, capable of measuring the density of the atmosphere anywhere within 10 kilometers. If a Fetri'sei was alerted when we crossed the airlock, by now the large airlock will be filling up with oxygen and argon to allow it to pass through, and this sensor here will register it. Let's see..."`

Author: I'd much rather keep this as a contraption, there's multiple precedents for this in the Keystone Research missions

> 			`	A few minutes later, pressurization finishes, and the interior door opens. You follow Dusk out of the airlock and find yourself in front of a large area, likely the same place the encounter with the Fetri'sei took place. "The moment of truth," exclaims Dusk, placing down a heavy scanner, similar to a short telescope. "This is a low-power laser system, capable of measuring the density of the atmosphere anywhere within 10 kilometers. If a Fetri'sei was alerted when we crossed the airlock, by now the large airlock will be filling up with oxygen and argon to allow it to pass through, and this sensor here will register it. Let's see..."`

Reviewer 5: Yeah, it looks like this is true even in 7948

			`	Dusk smiles. "The airlock is still empty. Looks like we can finally begin exploring the facility." When the rest of the group finishes crossing the airlock, you all start moving toward a curved board, placed equidistant from each of the entrances. After scrutinizing the writings on it for a few seconds, one of the Remnant exclaims, "A map!" Dusk starts discussing something with the rest of his team, but from your position you can't see what they're signing.`
			choice
				`	(Wait for them to finish their discussion.)`
				`	(Move closer to try to follow the conversation.)`
				`	(Try to get Dusk's attention.)`
			`	Dusk turns around to you. "So, <first>," he starts, "what do you think? This is indeed a map of the Builder complex. The team and I have split the exploration in four groups - one going to the armory, one visiting the underground facilities and another visiting someplace that roughly translates to 'part cycler.' If you want to come with me, we can choose between the general storage area and what seem to be some kind of elevated gardens. Or you could remain here to keep an eye on the laser scanner if you wish; the choice is yours."`
			choice

> 				`	"Let's go to the storage."`

Author: Ditto

			choice
				`	(Call Dusk with the radio.)`
					goto alarm
				`	(Ignore it.)`
			`	As the time passes, the light dims and blinks slower. It takes almost half an hour, but the yellow light finally turns off entirely.`
			`	Some time later the group returns, satisfied with what they've found inside. "It was an interesting trip for sure, Captain," Dusk tells you. "Did anything of note happen out here?"`
			choice
				`	(Tell him about the light.)`
					goto explain
				`	"Not much. It was a bit boring."`
			`	"I imagine so. You would have been more helpful in the complex together with me." After picking up the scanners, you begin to pass through the airlocks, three people at a time. After a few minutes, you're all back outside the dome, ready to finally return to <planet>.`
				goto ending

			label alarm

> 			`	As you tell him about the blinking light, he interrupts you and announces to the rest of the group that they are leaving. "The yellow light is an indicator that the battery is running out," he explains hurriedly. "I should have checked our equipment better. Soon, the starship airlock will be unwatched." Instead of going through a single airlock, the group splits, half going into one airlock and half into another. When you're finally all outside of the dome, the tense mood dissipates.`

Author: I don't think this one would help the phrase's flow

				accept
	on enter Lathia
		dialog
			`As soon as you enter the system, a dull hum coming from below the cockpit signals that the scanners have activated. While the Remnant equipment does its job, you take advantage of the opportunity to observe the mesmerizing colors of the neutron star, beautiful and terrifying at the same time; however, in a few minutes a beeping sound signals that the scan is complete, and you can return back.`

mission "Ka'het: Patir Mystery 12A"
	landing
	name "The Chanai station"
	description "Land in the station in Chanai to search for a way to return, then come back to <system>."
	passengers 5
	source "Builder Settlement"
	stopover "Chanai Structure"
	to offer
		has "Ka'het: Patir Mystery 11: done"
	on offer
		conversation
			`Like your previous trip, the onboard machinery beings transmitting the data right as you come into Patir's vicinity. The analysis job must have been quick: by the time you're near the landing area, you can see the Remnant standing outside the lab, waiting for your arrival. Dusk meets you in the cockpit right as you've finished safing the landed spacecraft.`

> 			`	"It does not look promising, <first>. Just as with the satellite, nothing beyond the most basic orbital information was recorded by the scanners we gave you, which makes it a complete failure. Maybe after in-depth analysis we could understand something about these beings even just with that, but it would still be nothing that could help us in our search.`

Author: The answers to this and to https://github.com/endless-sky/endless-sky/pull/9959#discussion_r1557300321 are connected, so replying here for both. That's kind of the idea - both moves are kind of out of desperation, in the sense that by this point they are far out of their depth and with no plan to follow. Both of these are essentially guesses, and as you've seen later they're both completely wrong.

Changed the wording as per your suggestion

			choice
				`	"Chanai seems to be a better option."`
					goto chanai
				`	"The new plan feels more viable to me."`
			label hack
			`	"Alright then. The next step will be deploying some satellites in the nearby systems to have a preemptive warning on the arrival of these beings, and so that we can approach a Fetri'sei as quickly as physically possible when the time comes. Reusing the sensors we put on your ship and coupling them with the spare parts we have in storage should be enough to assemble a few probes; meet us in the lab in a few hours, and we will be ready."`
				decline

			label chanai
			`	"Okay then. Most of the equipment we are going to need is the same equipment we used for Patir, so it will not take long to prepare. If you give us a hand, we should be ready to go in half an hour at most."`
				launch
	on enter Chanai
		dialog
			`The gargantuan station still sits far away from you, appearing as just a dot in the distance. A long range scan of the structure reveals enough of it for an initial assessment; after a quick discussion, the approach location has been chosen and you can proceed to dock.`
	on stopover
		conversation

> 			`The external coating of the structure appears pristine and untouched, much like a new spaceship. Orbiting far above a black hole, few asteroids have orbits capable of posing a threat to it, independently from its size. Next to you, one of the Remnant researchers is pointing a rather bulky scanner toward the station; after a few minutes, she indicates you a place from which the structure can be accessed, and you park your ship there.`

Author: Changed to `The external coating of the structure appears pristine and untouched, much like a new spaceship. Orbiting far above a black hole, next to no asteroid moves in orbits capable of posing a threat to it.` on top of your suggestions

			choice
				`	(Quickly take a sample and move on with your mission.)`
					goto moving
				`	(Inspect them closer.)`
			`	Growing from the vase almost up to the ceiling, the plant you choose to analyze stands at nearly twice your height; despite this, the trunk is only a few millimeters thick, thanks to the lack of gravity aboard the spacecraft. Slender, needle-like leaves are spread across the entire stem, contrasting its okra with a red tint.`
			`	Suddenly, as you try to take a better look at the fronds the entire trunk breaks in half with minimal pressure. Instinctively you try to pick up the upper segment before it flies away, but this only splits it into more parts; you decide to observe the resulting pieces to afar instead. To your surprise, the internal structure is completely empty, leaving a large hole inside the stem. You take a few pictures and a couple samples of both the surface and of the vase, moving then to the next room.`
			label moving

> 			`	The rest of the zone doesn't seem to be as interesting. A series of holes in a wall initially attract your attention, but they quickly prove too narrow to pass through and too profound to be able to gather anything on them; similarly, the corridor at the end of the area is slightly smaller than your suit, preventing you from continuing. As you're returning back to the rest of the group, however, you notice an unusual drawing you had missed the first time.`

Author: Yes, one room

			`	Attached between the various carvings, the canvas shows a series of points in a line with every color of the rainbow. While the lines are near the bottom of the mural, above them all shines what looks like a red sun, sending rays of light down to each of the colorful dots and surrounding them all in a crimson halo. Carefully passing your glove over its surface, you notice every part is composed of a different material, smoother and more refined than the walls around it.`

> 			`	Based on your oxygen levels, you need to meet up with the Remnant team as soon as possible; you follow Dusk's instructions up and down the structure until reaching a large open area, where you find them observing a vial of samples from the structure. Behind them, you see the start of the giant barrel that composes most of the station's structure.`

Author: What's the change?

				goto back

			label vial
			`	"Our ticket back, hopefully," Dusk replies with a half smile. "While everything we have seen is interesting in its own way, there is something in particular that gives me hope about finding a way out. We all need to start moving back right now - we have spent enough time studying this exhaust chamber, and the oxygen margins are starting to worry me - but as soon as we are on the way I'll show the reason for our excitement."`
			label back
			`	One at a time, the group enters back into the main tunnel, with Dusk as the back marker and you right in front of him. You notice the passage at the end doesn't show any blowtorch marks; it seems that the Remnant found it already open when they arrived here, which is odd for a port facing an outlet of something that big and powerful.`
			`	"While you were exploring that sector, we found a few more similar tunnels and split up into subgroups to explore each of them. This is how we have found the path to the barrel you have seen earlier, for instance, as well as many other peculiar areas. And, in one of them," he continues, clumsily passing you a data pad, "this is what we stumbled upon."`

> 			`	Looking at the screen, you see a photograph of two Remnant standing in front of a large mural, three times as tall as them and far wider. Despite the picture being poorly lit and rather confusing, you recognize it as a two dimensional starmap, carved into the wall of the station. Near the bottom right, a set of circles and unfamiliar markers surround one of the systems indicated.`

Author: As in poorly shot, hard to make out the shapes in it

				goto crystal
			label coincidence
			`	"Maybe. Without reading the text surrounding it, which is written neither in Builder nor any language I can recognize, we can only analyze what we can infer from the graphic itself - something far from ideal. But if we take it at face value, it would be the first sign toward a solution to our problem, if we can call it that, since we have arrived in this sector. And then, there's this."`
			label crystal
			`	Taking back his data pad, Dusk hands you the vial you saw earlier. Despite its small size, as you take it you can feel that it is rather weighty. Inside is a grainy dust interspersed with small, violet crystals of various shapes. "We found multiple rooms chock full of this material, all around the station. We haven't explored the entire complex, but if the pattern repeats we expect there to be something similar surrounding the entire the entire barrel."`

> 			`	"What is it made of?" you ask, looking at the particles moving inside the flask. They're similar to the regolith surrounding the Builder settlement orbiting Patir, but far less compacted. "Inconclusive. Perhaps the instrumentation of the lab will give us better results, but the tools we could bring with us returned only indecipherable noise. But then, we noticed a pattern: while the data was faulted, it was in the same exact way as the one you gathered in Lathia a few days ago. Whatever it is, it could be made of the same matter that composes those creatures. While we depart, I can use the scanners on your ship to confirm it without wasting any time."`

Author: Many of the examples in [this page](https://dictionary.cambridge.org/dictionary/english/faulted) seem to use it the same way. You're right about the noise

			`	"Oh, definitely! While the data we could gather was not as much as I hoped, we now have a rather clear connection between Patir and Chanai, as well as between the station and those peculiar beings. That, alone, is invaluable for our mission.`
			label next
			`	"During the trip back, we have evaluated the options at our disposal. As you may guess, the most obvious one would be to explore further into the station; however, it was not as attractive as previously thought. While it may have been used to influence Patir, it is clear that it sits now silent and powerless, as if missing something that can bring it back to life. We could find more about its design perhaps, but the chances of using it on our own within a reasonable time scale are low.`
			`	"We have enough consumables to last for many weeks still; we started rationing as soon as we arrived in this sector, and we are prepared to reduce them further if necessary. Nevertheless, we need to take a more direct approach if we want results. Approaching the species that inhabits this cluster is essential at this point, and doing so immediately would be advisable."`
			choice
				`	"What do you suggest?"`
					goto plan

> 				`	"Are you proposing a first contact of some sort?"`

Author: I would like to keep something akin to the original phrase here, given that said first contact is supposed to happen with a giant space rock

			`	"Given that you need to be able to shoot the target as quick as possible, you cannot return to the cockpit to keep an eye on the scan systems. So, we took the liberty of creating a small terminal here for you to keep an eye situation," he replies, indicating a small desk filled to the brim with Remnant machinery.`
			label entrance
			`	One by one, the scientists jump from your ship to the Fetri, moving toward a closed, circular hatch in between the two engines. As you watch them start the expedition, you sit down in the console they have prepared in the cargo bay, monitoring the movements in the system and any change in the slug. Though you can't see clearly what the five of them are doing, you distinguish someone inserting a rod inside a small aperture, followed by the hatch opening right after.`
			`	As the crew moves beyond your line of sight and the communications cease, your uneasiness about the mission keeps growing steadily. Time passes slowly as you wait for their return; as you split your attention between the various screens in front of you and the occasional glances out of the airlock, every minute seems to last an eternity. And then, it happened.`

> 			`	All above the surface of the asteroid-looking creature a burst of rose tinted sparks appears in an instant, quickly growing in number and coverage; a clear sign that it is about to jump away, and that the Fetri'sei is about to awaken. The expedition must end right now, or the Remnant may not survive the encounter.`

Author: Changed to `	Scanning over the monitors for the umpteenth time, you finally see that heightened energy levels have been detected inside the unknown creature. It is about to jump.` since the passage became to short and unexplanatory

			label depart
			`	Within minutes, the ship is starting the liftoff, the six of you standing in the cockpit with trepidation. "We will take care of the necessary analysis while you bring us to orbit. Hopefully we have been returned to the Milky Way, and not in yet another unreachable galaxy."`
				launch
	on enter
		conversation
			`The cockpit has become quite chaotic: the instruments the team brought with them keep buzzing and beeping during the whole trip, annoying you somewhat. After shutting down the main engines, someone taps on your shoulder to call your attention. "<first>, could you pitch the ship down a few degrees?" Dusk asks, with a calmer look than during liftoff. You reply with a nod, and move the stick forward slightly.`
			`	"There! Do you see it?" he continues, pointing at a spot in the sky with his hand. Between the mass of stars in front of you, a pattern leaps to the eye: two set of three stars, right next to each other, forming a prism-like structure. "We did it. We are back, finally."`
	on complete
		payment 370000
		event "patir mystery 2 timer" 45

> 		conversation

Author: Done

			label finish
			`	The two of them accompany you to the <ship>, sitting alone in the now-quiet spaceport of <planet>. The cargo bay has been emptied of vast array of instruments that had been lying around for a while, and you guess the same has been done with the cockpit. As you walk once again up the stairway, the Remnant bid you farewell. "We have transferred <payment> as a way to express our gratitude. Of course, their worth is incomparable with the value of what you have done for us; without you, we would have never made it out alive. Goodbye for now, and may the Embers aid in your pursuits."`

event "Patir returns"
	system Patir
		pos -38 553

mission "Ka'het: Patir Mystery 13B"
	name "Return to <planet>"
	description "As the Patir black hole appears to have returned to the Milky Way, it's time to return to <planet> together with Dusk."
	source "Builder Settlement"
	destination "Aventine"
	to offer
		has "Ka'het: Patir Mystery 12A: done"
	on offer
		event "Patir returns"
		conversation

> 			`The time has come: the walk toward the lab is quick and straightforward, and you have cover the whole distance in a matter of minutes. The rest of the group, preparing their backpacks inside of the outpost, greets you and starts explaining the outline of the mission being planned.`

Author: Rewritten:
`The time has come: the walk toward the lab is quick and straightforward, and you cover the whole distance in a matter of minutes. Peeking through the hatch's window, you see the rest of the group preparing their backpacks, with prepacked toolboxes surrounding them in every direction. You wave through the glass, and they return the greeting.`
			`	You move the airlock's heavy door to pass through, and wait for the room to be re-pressurized before walking in. By the time the hatch unlocks, however, the Remnant are gone, hastily leaving their bags on the floor.`
			`	As you take a few steps inside the lab, you turn around to find them all glued to the windows with an awestruck gaze. The sky has again turned black, and a few dim lines are already starting to appear in the distance.`
			`	While stars and galaxies zoom in the night sky, a few of the Remnant run around the lab, turning on various instruments. However, most of them remain with their faces stuck to the glass. The lights start converging toward a single point, forming a large, but rapidly shrinking mass in the background.`
			`	The stars begin to slow down before settling into their final positions. After all the time you have spent here, it isn't difficult to recognize that the sky has changed again. Next to you, the rest of the team is already analyzing the data gathered; Dusk seems to be writing down something from a chronometer.`

			choice
				`	"What is going to happen to Patir now?"`
					goto patir
				`	"Wait, what if we weren't around Patir when it came back?"`
			`	"The black hole would have departed without us, and we would have been still stuck in that sector - only this time without a laboratory to support us, nor most of our consumables. The people down here perhaps could have been able to send Patir back there to pick us up, but by then we would have run out of supplies long before, and perished between the stars. We were lucky that things went the way they did, all things considered. Our research in those systems was supposed to bring us back, and yet we were just chasing rainbows, endangering us further every step we took.`
				goto next

			label patir
			`	"It is difficult to say. Given what we have observed, it will probably disappear once again after the effects of our energy burst; when that will happen or where it will go is beyond our current knowledge. Those same systems in which we risked remaining stuck could have now become inaccessible, and our data may be the only we will ever gather on them.`
			label next

> 			`	"At this point in time, exploring Patir would be a gable, and an extremely risky one at that. The next phase will have to be operated from a distance, in order to understand the way that system works and how Ssil Vida controls it. How much that will take, I can't say; maybe weeks, or months, or even years. But then, and only then, we will come back there and start digging deeper."`

Reviewer 2: ```suggestion
			`	"At this point in time, exploring Patir would be a gamble, and an extremely risky one at that. The next phase will have to be operated from a distance, in order to understand the way that system works and how Ssil Vida controls it. How much that will take, I can't say; maybe weeks, or months, or even years. But then, and only then, we will come back there and start digging deeper."`
```

	"reputation: Ka'sei" = 1

event "patir mystery timer"

ship "Cloak Check"
	attributes
		"heat dissipation" 1
		"hull" 3
		"mass" 100
		"drag" 5
		"automaton" 1
		"thrust" 100
		"turn" 6000
	outfits
		"Cloak Checker"
	turret 0 0

outfit "Cloak Checker"
	category "Turrets"
	thumbnail "outfit/unknown"
	"turret mounts" -1
	weapon
		"velocity" 10
		"lifetime" 1
		"reload" 1
		"blast radius" 20
		"hull damage" 5
		"no damage scaling"
		"range override" 1000000

> 		"turret turn" 360

Reviewer 6: If we update the `on encounter` NPC trigger to work on your escorts in addition to your flagship, or provide a separate trigger that works on your escorts, then we could replace these with just a normal NPC with an `on encounter` node.

			choice
				`	"I'm in."`
					accept
				`	"What if they detect us while inside the structure?"`
				`	"Sorry, but I don't think it is a good idea to return there."`
					decline
			`	"It is a possibility, yes. It is also quite problematic because communication between the inside and the outside of the structure is impossible. I think the best course of action would be to place a scanner that controls the large airlocks on the upper side of the dome, the same ones the Ka'sei used last time. If I am right, it should give us around a quarter of an hour to get back in the airlock and escape."`
			choice
				`	"Alright, let's do it."`
					accept
				`	"Sorry, but I don't think it is a good idea to return there."`
					decline

	on accept
		payment 100000
		set "remnant: dusk busy"
	on abort
		# Do nothing.
	on fail
		dialog
			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`
	npc save
		government "Test Dummy"
		personality heroic nemesis unconstrained
		system "Patir"
		fleet
			variant

> 				"Cloak Check" 10

Author: "save" means that the mission fails if any of the NPCs die

			`	"Because of that, before we can start our mission, you are going to need to make a stopover at <stopovers> - I was able to convince the prefects on Viminal to reallocate one of the outposts that they had just delivered there to our mission, given it has higher priority status compared to the mining expeditions they are setting up."`
			`	You help Dusk transport the chosen instruments to your ship; you also make sure to leave enough space for the facilities you're going to take from <stopovers>. As you're walking back toward the ship's controls, Dusk reminds you to bring as many ramscoops with you as possible - you're going to need them in order to complete the mission plan while cloaked.`
				accept
	on stopover
		dialog

> 			`Dusk's coordinates indicate the border of a large canyon near the equator of <stopovers>. Unlike most cracks on the surface, this one shows signs of geological activity: you only narrowly avoid a stream ejecting from a geyser in the canyon as you pass overhead. As you land, a row of Remnant structures are clearly visible running parallel to the cliff.`

Author: TIL that's a thing

> 			`	The local crew assist in loading the modular laboratory split across three pallets; with them arrives a separate package, containing a surveillance satellite to be deployed near Patir to scan the Ka'sei from orbit.`

Reviewer 6: Maybe add a reminder that you should be cloaked when you enter the system.

	on abort
		# Do nothing.
	on fail
		dialog
			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`

	npc save
		government "Test Dummy"
		personality unconstrained heroic nemesis
		system "Patir"
		fleet
			variant
				"Cloak Check" 10

mission "Ka'het: Patir Mystery 8B"

> 	invisible

Reviewer 6: This didn't offer because Babiali has no spaceport and this will only offer in a spaceport right now.
```suggestion
	landing
	invisible
```

	source "Babiali"
	destination "Builder Settlement"
	to offer
		has "Ka'het: Patir Mystery 8A: offered"
	on enter Patir
		dialog
			`The probe slowly moves away from your ship, opening its external panels and the instrumentation; as you watch it turn into a dim point of light, the Remnant team confirms that it is functioning correctly. Time to land on the asteroid.`

mission "Ka'het: Patir Mystery 9"
	landing
	name "Explore the neighborhood"
	description "Visit every system around <system> you can reach, then return to Dusk in <system>."
	source "Builder Settlement"
	waypoint "Athiri"
	waypoint "Chanai"
	waypoint "Ghila"
	waypoint "Lathia"
	waypoint "Maithi"
	waypoint "Mitera"
	waypoint "Sarifa"
	waypoint "Thepa"
	to offer
		has "Ka'het: Patir Mystery 8A: done"
	on offer

> 		event "Patir moves"

Reviewer 6: You can do `event "Patir moves" 0` to cause this to run instantly. Right now, if the player pulls up the map before they depart, they'll still appear to be in the Ember Waste. You could probably use an `action` node to call the event at the moment that the system moves so that opening up the map before and after that point in the conversation is when the move occurs.

			choice
				`	(Accept.)`
					goto tunnel
				`	"Do you have any information about this section?"`
			`	"Not much. We only had time for a brief scan during the approach, but based on the volume of these sectors alone, it shouldn't take you more than a couple hours to explore them.`
			label tunnel
			`	"Whenever you reach a new area, take samples and pictures of anything you deem interesting. Never forget to keep an eye on your oxygen: each of us will consume it at slightly different rates, and we should start the trip back as soon as someone has less than a third of a tank left. May the Embers help your search, Captain, and see you soon!"`
			`	Moving in the zero gravity station is quick and relatively easy; despite its length, it only takes you a few minutes to reach the end of the right tunnel. All you can find, however, is a closed hatch. A few tries at forcing it open only serve to confirm that it is locked shut, and you decide to go back and try the other tunnel. This time, your search turns up results, and you end up in a dark, somewhat cramped room.`

> 			`	The room is lavish and dazzling, the walls covered with intricate carvings. It's a striking difference from the gray and slightly coarse material everywhere else in the station, pushing you to gather a few samples to analyze later. To an extent, it reminds you of the Art Deco paintings of ancient humanity. Around the room, alien plants of varying height and shape paint every corner with a reddish hue.`

Author: > art student

I am an aerospace engineering student

I very much think it would make no sense to write it as "art decoration", and I would also prefer to keep the capitalization given that we are talking about a specific artistic style

			`	"While you were exploring that sector, we found a few more tunnels and split up into subgroups to explore each of them. This is how we found the path to the barrel, for instance, as well as many other peculiar areas. And, in one of them," he continues, clumsily passing you a data pad, "this is what we stumbled upon."`
			`	Looking at the screen, you see a photograph of two Remnant standing in front of a large mural, three times as tall as them and far wider. Despite the picture being poorly lit and rather blurry, you recognize it as a two-dimensional starmap, carved into the wall of the station. Near the bottom right, a set of circles and unfamiliar markers surround one of the systems indicated.`
			`	"Look at the orbital parameters of that system. It is, without a shadow of doubt, indicating Patir. A system they should not be aware of, if it only just arrived here, and one they were apparently interested in."`
			choice
				`	"It could be a coincidence."`
					goto coincidence
				`	"For what purpose?"`

> 			`	"I would also like to know that. I cannot recognize the language in the text, and decrypting it would be an enormous task - one that would take years of focused effort from the whole community, provided we can gather enough data on it. We must base our decisions only on what we can infer from the graphic itself, which is far from ideal. And then, there is this."`

Author: Changed to `one that would require a vast amount of text to analyze, as well as years of focused effort from the whole community.`

			label lab
			`	"Meet us in the lab in a few hours, and we will have everything ready for the trip. With a bit of luck, most of the instruments we used on Chanai will be a good fit for this mission as well."`
		
mission "Ka'het: Patir Mystery 12B"
	name "Satellite deployment"
	description "Deploy two probes in <stopovers> to keep an eye on incoming space rocks."
	source "Builder Settlement"
	passengers 1
	cargo "Remnant Probes" 3
	waypoint "Mitera"
	waypoint "Thepa"
	to offer
		has "Ka'het: Patir Mystery 11: done"
		not "Ka'het: Patir Mystery 12A: accepted"
	on offer
		conversation
			`You find the team waiting for you in the lab. Next to them are two mechanical crates, roughly the size of a person. Most of the clutter around the room has been cut down considerably; these satellites alone must have used the largest part of their reserves. "Here they are, Captain," Dusk states proudly. "Two autonomous surveillance probes, capable of scanning an entire system without any input and slowly beaming the results back to us. Their capabilities are far from what our main fleet is capable of back home, but considering our situation, they should be more than enough."`

> 			`	"We have settled on their destinations: the stellar black holes of <waypoints>. Once there, they will be far enough to give us appropriate warning, but close enough to only pick up asteroids heading towards Patir. If they do, that will be our signal to depart and intercept them right as they come in."`

Author: Presumably the "entangled particle pair" mentioned in Remnant Defense 3

				accept
	on enter "Mitera"
		dialog
			`As the satellite leaves your ship, you watch it move further away and unfurl its solar arrays. The probe quickly disappears in the blackness of space.`
	on enter "Thepa"
		dialog
			`Dusk quickly reaches for the console, initiating the satellite deployment procedure you've become accustomed to. It doesn't take long for it to complete, and in a few minutes you're ready to leave the system once again.`

ship "Asteroid Blocker"
	attributes
		"heat dissipation" 1
		"hull" 3
		"mass" 100
		"drag" 5
		"automaton" 1
		"thrust" 100
		"turn" 6000
	outfits
		"Fuel Removal"
	turret 0 0

outfit "Fuel Removal"
	category "Turret"
	thumbnail "outfit/unknown"
	"turret mounts" -1
	weapon
		"velocity" 1000
		"lifetime" 10
		"reload" 100000
		"blast radius" 200000
		"safe"
		"no damage scaling"
		"range override" 10000000
		"homing" 4
		"tracking" 1
		"acceleration" 1.5
		"drag" 0.2
		"turn" 12
		"fuel damage" 500
		"turret turn" 360

mission "Ka'het: Patir Mystery 12C"
	landing
	name "Ka'sei hacking"
	description "Taking advantage of the arrival of one of the asteroids, berth with a downed Ka'sei to gather data from its internal computers."

> 	passengers 5

Author: Added

event "no Patir spaceport and no Fetri spawn"
	system Patir
		remove fleet "Ka'sei"
	planet "Builder Settlement"
		remove spaceport

event "Patir spaceport returns and Fetri spawn"
	system Patir
		add fleet "Ka'sei" 300
	planet "Builder Settlement"
		spaceport `The Remnant laboratory merges oddly well with the rest of the environment, similar in color and design to the Builder structures around here. The spaceport amounts only of a flat strip of regolith near the settlement, hastily cleaned of the debris that covers most of the terrain.`

mission "Ka'het: Patir Mystery 12D"
	landing
	invisible
	source "Builder Settlement"
	to offer
		has "Ka'het: Patir Mystery 12B: done"

> 	npc assist

Author: What do you mean?

			choice
				`	"Are we back in the Milky Way?"`
				`	"How long was that?"`
					goto time
				`	"Where are we now?"`
			`	"I cannot say yet," Dusk replies hurriedly. "Quick, put your EVA suit back on. We need to depart as soon as possible, all of us together. I do not know how much time we have." You hastily connect your helmet back to the life support and follow them out of the airlock, trying to keep up as best as you can.`
				goto depart

			label time
			`	"Around four minutes," Dusk replies hurriedly. "But we will talk about that another time. We need to depart as soon as possible, all of us together. I do not know how much time we have." You hastily connect your helmet back to the life support and follow them out of the airlock, trying to keep up as best as you can.`
			label depart
			`	It takes only a few minutes to launch the <ship>, the six of you standing in the cockpit with trepidation. "We will take care of the necessary analysis while you bring us to orbit. Hopefully we have been sent to the Milky Way, and not to yet another unreachable galaxy."`
				launch
	on enter
		conversation "end of Patir 2 - on enter"

	on complete

> 		payment 370000

Reviewer: This one wasn't changed.
```suggestion
		payment 1570000
```

	"reputation: Ka'sei" = 1

event "patir mystery timer"

ship "Cloak Check"
	attributes
		"heat dissipation" 1
		"hull" 3
		"mass" 100
		"drag" 5
		"automaton" 1
		"thrust" 100
		"turn" 6000
	outfits
		"Cloak Checker"
	turret 0 0

outfit "Cloak Checker"
	category "Turrets"
	thumbnail "outfit/unknown"
	"turret mounts" -1
	weapon
		"velocity" 10
		"lifetime" 1
		"reload" 1
		"blast radius" 20

> 		"hull damage" 5

Author: No, as far as I am aware the NPC won't fire something that will make its hull negative unless it does so with blast radius

				goto proposal

			label "atmospheric data"
			action
				payment 300000
			`	One of the other Remnant shows you a hologram of the system, with what looks like the Builder settlement you visited earlier highlighted by a yellow light. "First of all, we have transferred <payment> to your logistics account for saving the data on the dome's atmosphere. It is only thanks to you that we were able to learn more about what kind of atmosphere the Builders may have lived in."`
			choice
				`	"Thank you."`
					goto skipdialog
				`	"Can you tell me more about that?"`
			`	"It seems their atmosphere may have been similar to ours - the air in the settlement we visited was almost evenly split between oxygen and argon: breathable by human standards."`
			`	"We could have removed our helmets after the airlock?" you ask.`

> 			`	"It is a bit more complicated than that. We found that the pressure inside the complex was much, much lower than what we are used to - less than a sixteenth of an atmosphere. On most Earth-like planets, you would only find such pressures at over 20,000 meters. There are two possibilities: either the Builders were born in a thin atmosphere, or the life support system failed some time ago and the structure has been slowly losing air ever since. We hope to find the answer soon enough."`

Author: They use meters per second in Visit to Paradise and cubic meters in a number of instances, so I would say it's likely

			label entrance
			`	It doesn't take long to reach the airlock ring in front of the dome. The airlock you used last time is still open, and its control panel is visible next to it. Dusk, following your gaze, quickly stops you. "I doubt it would be a good idea to pass through the same airlock we used last time, <first>," he says. "It looks like the structure's automated systems patched up the hole left by the Ka'sei, but I would never trust it as a pressure vessel at this point. Plus, I count at least six more airlocks we could use."`
			`	He and another member of the group move to the opposite side of the control station next to the airlock, then flip one of the switches on the panel. Just like last time, a door opens next to them, revealing another on the opposite side of the small room. There is space only for three crew members at a time; you, Dusk, and another Remnant enter the airlock, while the other three wait their turn. The door closes, and Dusk flips the switch to begin the pressurization procedure. Suddenly, the lights inside the module turn on, emitting a bright, azure light.`
			choice
				`	"Did you do that?"`
				`	"What happened?"`

> 			`	"I am not sure. Wait." Moving to the opposite side of the airlock, he uses a crowbar to open another hidden control station, which slides out of the wall to reveal a rather large screen scrolling through some alien writing. After reading through it with some difficulty, Dusk breathes a sigh of relief and pulls it back to its original location. "I think it is nothing," he adds, "at least I hope so. From what I could gather, the lights being switched on were a normal, timed event in the commanded startup sequence - why they they did not turn on last time is beyond me, but nothing seems to be out of the ordinary. Perhaps they were just broken."`

Author: As mentioned in Patir 1, the Remnant have partially translated the written Builder language

> 			`	"I am not sure. Wait." Moving to the opposite side of the airlock, he uses a crowbar to open another hidden control station, which slides out of the wall to reveal a rather large screen scrolling through some alien writing. After reading through it with some difficulty, Dusk breathes a sigh of relief and pulls it back to its original location. "I think it is nothing," he adds, "at least I hope so. From what I could gather, the lights being switched on were a normal, timed event in the commanded startup sequence - why they they did not turn on last time is beyond me, but nothing seems to be out of the ordinary. Perhaps they were just broken."`

Reviewer: So it's written on the outside? Might be worth mentioning that further ahead

> 			`	"I am not sure. Wait." Moving to the opposite side of the airlock, he uses a crowbar to open another hidden control station, which slides out of the wall to reveal a rather large screen scrolling through some alien writing. After reading through it with some difficulty, Dusk breathes a sigh of relief and pulls it back to its original location. "I think it is nothing," he adds, "at least I hope so. From what I could gather, the lights being switched on were a normal, timed event in the commanded startup sequence - why they they did not turn on last time is beyond me, but nothing seems to be out of the ordinary. Perhaps they were just broken."`

Author: It's written in the `rather large screen scrolling through some alien writing`, so I felt it was implied. I can explicitize it more if needed

> 			`	A few minutes later, pressurization finishes, and the interior door opens. You follow Dusk out of the airlock and find yourself in front of a large area, likely the same place the encounter with the Fetri'sei took place. "The moment of truth," exclaims Dusk, placing down a heavy scanner, similar to a short telescope. "This is a low-power laser system, capable of measuring the density of the atmosphere anywhere within 10 kilometers. If a Fetri'sei was alerted when we crossed the airlock, by now the large airlock will be filling up with oxygen and argon to allow it to pass through, and this sensor here will register it. Let's see." A tense silence ensues as the machine boots up.`

Author: An airlock works by equalizing the pressure between the two sides of the hatch, filling it up with the atmosphere of the pressurized side. It's not for the survival of the Ka'sei, but for the (intended) people inside the complex.

			`	If the depot seemed large from the outside, it's even more impressive seen from the inside: massive, sturdy shelves tower left and right, surrounded by odd machinery and cylindrical containers. While you're still studying the structure, Dusk calls you over. "Look here, <first>," he says, indicating the lower shelves in front of him. "What do you see?" It isn't easy to inspect the shelves with just your helmet lights, but you notice they're all empty. "This is not what I would expect from a settlement that used to be inhabited."`
			choice
				`	"What do you think happened here?"`
					goto theory
				`	"Maybe the rest of the depot will have something."`
					goto samples
				`	"Is it possible that they just emptied the storage when they left?"`

> 			`	"Maybe. But look at how pristine this place is. There is no dirt or dust anywhere I can see, and everything seems to be exactly where it should. It seems more as if it was never inhabited in first place. There are far too many unknowns about Patir and this settlement to be certain of anything, though." He pulls another instrument from a waist pouch, then begins gathering samples from around the depot, both from the ground and on the shelves. The procedure takes a few minutes; he then puts the samples in one of the cylinders on his back and starts moving to the next room, signing for you to follow him.`

Author: This one feels unnecessarily repetitive - " to be exactly where it should be"

			label plan
			`	"Even if there are no plants, there must be containers with seeds and fertilizers somewhere around here. We have already wasted a lot of time coming up here; if you go look for those while I gather samples from the abandoned hydroponics cultures, we will cover more terrain in the time we have left." As Dusk begins to prepare the sampling vials and the rest of the equipment, you walk toward the back of the "fields." You enter a series of tall rooms parallel to the hydroponics, none of which contain anything of note - all except the last room, which contains numerous hexagonal containers stacked on top of each other.`
			choice
				`	(Call Dusk.)`
					goto call
				`	(Open one of the boxes.)`
			`	You take one of the boxes from the top of the stack. It seems very heavy for its size, but with the help of the almost negligible gravity of the asteroid, you eventually succeed in taking it down to the floor. Opening it is even harder, as the box seems to have been sealed shut hermetically. After a lot of struggle, it opens to reveal a mass of spherical, light blue seeds tightly packed together.`

> 			`	Dusk reaches you shortly thereafter, but he doesn't seem too happy. "I have few pressurized containers with me, <first>. We cannot expose the seeds to the vacuum of space without risking killing them, and we are going to need as many chances as possible to make something grow." Taking another box from the stack, he continues, "We ought to leave this one here for a future expedition and grab another instead."`

Author: Because he finished gathering samples by the time you find and pry open the box

			choice
				`	"What is the next move?"`
					goto next
				`	"What is in that crate?"`
			`	"Oh, those? They are a Builder sidearm - a weapon capable of emitting short range electrical bursts strong enough to incapacitate or kill your target. We discovered a few hundred of them inside some bunkers in the Graveyard long ago. If you would like, I would not mind misplacing one if you are interested in them, as a token of thanks for all your help here."`
			scene "outfit/ka'het taser"
			choice
				`	"Thank you. How do I use it?"`
					goto outfit
				`	"I'd rather be paid in credits instead."`
			`	"Oh, I have already transferred 50,000 credits to your account. I need to organize some things for our next mission; if you meet me in the spaceport a few hours from now, I should have everything ready."`

> 				accept

Author: It's an endpoint to avoid reaching `label outfit`, decline or defer would have the same result

> 				accept

Reviewer: I think the style in this case is to use decline.
```suggestion
				decline
```

				accept
	on stopover
		dialog
			`Dusk's coordinates indicate the border of a large canyon near the equator of <planet stopovers>. Unlike most cracks on the surface, this one shows signs of geological activity: you only narrowly avoid a stream ejecting from a geyser in the canyon as you pass overhead. As you land, a row of Remnant structures are clearly visible running parallel to the cliff.`
			`	The local crew assist in loading the modular laboratory split across three pallets; with them arrives a separate package, containing a surveillance satellite to be deployed near Patir while cloaked.`
	on abort
		clear "remnant: dusk busy"
	on fail
		dialog
			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`

	npc save
		government "Test Dummy"
		personality unconstrained heroic nemesis
		system "Patir"
		fleet
			variant
				"Cloak Check" 10

mission "Ka'het: Patir Mystery 8B"
	landing
	invisible
	source "Babiali"
	destination "Builder Settlement"
	to offer

> 		has "Ka'het: Patir Mystery 8A: offered"

Reviewer: Ah, right.

			choice
				`	"What is your reason for not having a symbiont?"`
					goto reason
				`	"Do you have a symbiont, Dusk?"`
					goto dusksymbiont
				`	(Just finish the dinner and leave.)`
			`	The rest of the dinner passes rather quickly, and within half an hour you're ready to return to your ship. At the entrance, Dusk approaches you.`
				goto next
			label dusksymbiont
			`	"I do have one - right under the cervical vertebrae, at the top of my back. Being the lead researcher of this mission, there was not much choice left; removing it at this point wasn't an acceptable option, and switching positions this late into the program would have been much more harmful than increasing the reserves planned to the right level." As the dinner comes to an end, you and Dusk move back near the entrance.`
				goto next

			label reason
			`	"For me, the reason is medical. My body rejected the symbiont quite violently shortly after the initial operation, to the point that even attempting install another would be very risky. But if you ask around, the causes are quite varied." As the dinner comes to an end, you and Dusk move back near the entrance.`
			label next

> 			`	"Well," Dusk replies, "our current plan is to remain at the outpost for one or two months, exploring the Builder settlement and studying everything it has to offer. Do not go too far in the meanwhile, though - emergencies can always happen, and you are supposed to be our ride back when time comes." He pauses for a few seconds to think. "If you are interested, perhaps you could remain here with us to help with the research. You would need to sleep in your starship, of course, and the supplies would be slightly more limited, but neither issue is a showstopper. It is up to you, <first>."`

Author: I've simplified to just `"Our current plan is to remain at the...`

			`	You can't tell how much time has passed since the phenomenon began, but it finally starts to show signs that it's coming to an end: the streaks become shorter and last longer in the sky, and the number of stars you can see becomes less and less. You begin noticing more and more details; some streaks seem to end before reaching the horizon, while others follow a curved or even cyclical pattern around Patir. Eventually, the stars come to a standstill, and everything returns to normal.`
			`	Still dizzy from the experience, you carefully walk back down outside to speak with the Remnant. By the time you've reached them, they seem to have recomposed themselves and have started studying some sort of holographic stellar map.`
			action
				event "Patir moves" 0
			choice
				`	"What was that?"`
					goto what
				`	"Is everyone okay?"`
			`	"We are all shaken by the experience, but we have much bigger problems to think about now," Dusk replies. "Something is not right with the sky."`
				goto constellation
			label what
			`	"I wish I knew. And the worst may have yet to come," Dusk replies. "Something is not right with the sky."`
			label constellation

> 			`	He points at the projector. "Do you see those two triangular formations near the top of the star map? That is something we called the Ember Prism - a constellation of six stars of similar magnitude, located far from the Wastes and close to each other. While it is not visible from Earth, it should be recognizable from almost every system here, including Patir; but no longer. And that is not all. Nothing in this star map can be found anywhere in the sky." He stops for a second, giving a worried look to the rest of the team. "We are no longer in the Ember Wastes."`

Author: Without going into specifics of what's happening (I can do it in DMs if needed), Patir's behavior is meant to be a (sort of) natural phenomena that the Remnant have run into and can interfer with

event "Patir moves"
	system Patir
		pos 10059.5 10.5
		remove haze
		add hazard "Beyond Patir Shield Eraser" 1
		remove hazard "Ember Waste Base Heat" 100
		remove hazard "Ember Waste Base Storm" 9000
	planet "Builder Settlement"
		spaceport `The Remnant laboratory merges oddly well with the rest of the environment, similar in color and design to the Builder structures around here. The spaceport consists only of a flat strip of regolith near the settlement, hastily cleaned of the debris that covers most of the terrain.`

mission "Ka'het: Patir Mystery 10"
	landing
	name "Gather data"
	description "Board the Remnant satellite in orbit around <system> to learn more about the odd creatures around here."
	passengers 1
	source "Builder Settlement"
	to offer
		has "Ka'het: Patir Mystery 9: done"
	on offer
		conversation
			`By the time you land, the Remnant are already discussing their analysis of the data recorded and transmitted by your ship. When they notice you, they abandon their current conversation and move closer to speak with you in-person.`

> 			`	They listen carefully as you detail your journey: the Ghila ringworld, the one-way wormholes, the Chanai station, and the many oddities in each of these systems. From the questions they ask you, they seem to be particularly interested about the beings roaming this sector. By the time you've finished recounting the last events, almost an hour has passed.`

Author: While not described in game yet, it's not meant to be a Quarg ringworld. It is intended as a [Penrose machine](https://en.wikipedia.org/wiki/Penrose_process), harvesting energy from the rotational momentum of the Ghila black hole below it (perhaps using [electrodynamic tethers](https://en.wikipedia.org/wiki/Electrodynamic_tether)) - a last ditch effort from a species which had practically no other way of generating power anymore

> 			`	They listen carefully as you detail your journey: the Ghila ringworld, the one-way wormholes, the Chanai station, and the many oddities in each of these systems. From the questions they ask you, they seem to be particularly interested about the beings roaming this sector. By the time you've finished recounting the last events, almost an hour has passed.`

Author: changed to `asteroid-looking creatures`

			`	Moving through the opening, you reach a long, cylindrical corridor, just as dark as the room you just came from. Based on the many handles and scaffolding attached to the surfaces around you, you guess this part of the station used to be inhabited. After following the group through the tunnel for a few minutes, you reach an intersection: the corridor splits into two lateral openings, each of them seemingly as small and endless as the one you're in.`
			`	"We only have a limited amount of time for this expedition, <first>. You have seen us work a few times now; can you explore the two tunnels on your own and reach us when you have completed it?" Dusk asks. "We will remain in radio contact the entire time."`
			choice
				`	(Accept.)`
					goto tunnel

> 				`	"Do you have any information about this section?"`

Author: Added the option, with the reply:
`	"Well, we have our own tunnels to go through," he replies half-seriously. "We have a lot of ground to cover, and only so much air in our suits. The safest way to increase our reach without staying here for weeks is by splitting up, like we did when we first explored the Builder settlement a few weeks ago.`

			`	"We have settled on their destinations: the stellar black holes of <waypoints>. Once there, they will be far enough to give us appropriate warning, but close enough to only pick up asteroids heading towards Patir. If they do, that will be our signal to depart and intercept them right as they come in."`
			choice
				`	"Understood."`
					accept
				`	"Is there a specific orbit I should deploy these probes in?"`
			`	"Oh, there is no need to worry about that. Their scanners are easily powerful enough to cover the whole system. Their power and communication capabilities are far more limited, but for the time we need them to last, this should not be an issue. Bring me to that system, and I will handle the rest quickly."`
				accept
	on enter "Mitera"
		dialog
			`As the satellite leaves your ship, you watch it move further away and unfurl its solar arrays. The probe quickly disappears in the blackness of space.`
	on enter "Thepa"
		dialog

> 			`Dusk quickly reaches for the console, initiating the satellite deployment procedure you've become accustomed to. It doesn't take long for it to complete, and in a few minutes you're ready to leave the system once again.`

Author: It should still make sense, as you've seen this happen either way already at the beginning of the mission chain (after taking it from Babiali, right before Patir moved)

			label move
			`	"Inside, we will search for the memory banks of the spacecraft. After a quick selection based on the data we can scrape, we will take the few that contain information concerning our research and extract their contents. Understood?"`
			choice
				`	(Nod.)`
					goto entrance
				`	"What about me?"`
			`	"Given that you need to be able to fire as quick as possible, you cannot return to the cockpit. So, we took the liberty of creating a small terminal here to keep an eye on the situation." He points to a small desk in the cargo bay filled to the brim with Remnant machinery.`
			label entrance
			`	One by one, the scientists jump from your ship to the Fetri, moving toward a closed, circular hatch in between the two engines. As you watch them start the expedition, you sit down at the makeshift console, monitoring movements in the system as well as any changes to the slug. Though you can't clearly see what they are doing, you distinguish someone inserting a rod inside a small aperture, followed by the hatch opening right after.`

> 			`	As the crew moves beyond your line of sight and communications cease, your unease grows steadily. As you split your attention between the various screens in front of you while glancing out of the airlock, every minute seems to last an eternity.`

Author: Reworded to `You split your attention between the various screens in front of you while glancing out of the airlock, and the clock on your suit's wrist keeps on ticking slowly.` to make it less on the nose

				`	"Where are we now?"`
			`	"I cannot say yet," Dusk replies hurriedly. "Quick, put your EVA suit back on. We need to depart as soon as possible, all of us together. I do not know how much time we have." You hastily connect your helmet back to the life support and follow them out of the airlock, trying to keep up as best as you can.`
				goto depart

			label time
			`	"Around four minutes," Dusk replies hurriedly. "But we will talk about that another time. We need to depart as soon as possible, all of us together. I do not know how much time we have." You hastily connect your helmet back to the life support and follow them out of the airlock, trying to keep up as best as you can.`
			label depart
			`	It takes only a few minutes to launch the <ship>, the six of you standing in the cockpit with trepidation. "We will take care of the necessary analysis while you bring us to orbit. Hopefully we have been sent to the Milky Way, and not to yet another unreachable galaxy."`
				launch
	on enter
		conversation "end of Patir 2 - on enter"

	on complete
		payment 1570000
		event "patir mystery 2 timer" 45
		set "initial patir exploration"
		conversation "end of Patir 2 - on complete"

> event "Patir returns"

Author: The description only mentions the lab and that one will still be there, so I'd leave it tbh

			`	You move the airlock's heavy door to pass through, and wait for the room to be re-pressurized before walking in. By the time the hatch unlocks, however, the Remnant are gone, hastily leaving their bags on the floor.`
			`	As you take a few steps inside the lab, you turn around to find them all glued to the windows with an awestruck gaze. The sky has again turned black, and a few dim lines are already starting to appear in the distance.`
			`	While stars and galaxies zoom in the night sky, a few of the Remnant run around the lab, turning on various instruments. However, most of them remain with their faces stuck to the glass. The lights start converging toward a single point, forming a large, but rapidly shrinking mass in the background.`
			`	The stars begin to slow down before settling into their final positions. After all the time you have spent here, it isn't difficult to recognize that the sky has changed again. Next to you, the rest of the team is already analyzing the data gathered; Dusk seems to be writing down something from a chronometer.`
			action

> 				event "Patir returns" 0

Author: Derpy asked to put this here so that the map will show the right position before and after patir moves

	choice
		`	(Wait for him to speak.)`
			goto wait
		`	"Do you know how we came back to this galaxy?"`
		`	"What happened while we were gone?"`
	`	"Oh. I thought you had been told already." He sits down in front of you. "It was us that brought Patir back here. After your departure, it did not take long for our long-range sensors to pick up that the black hole had disappeared, along with you and the team. Our data was scarce, but we knew one, very important thing: if Patir had appeared here once, there must have been a way to do that again.`
		goto explain
	label wait
	`	"If your description is accurate, it means Patir is not another mouth to a pocket universe, but instead a transporter to another galaxy. Or universe, for all we know. In retrospect, there were enough reasons to doubt that first theory, but we were far too busy trying to save you to look into it seriously." Seeing your confused expression, he continues. "<first>, it was us who brought Patir back here.`
	label explain

> 	`	"We knew Postverta was involved with Patir's initial arrival, and together with the team there, we discovered that the energy burst that had been released when Ssil Vida first appeared had no correlation with the systems controlling its pocket universe. So, its activation or deactivation would have no effect on Patir, and we had to start looking somewhere else.`

Reviewer 7: Sorry if I'm misinterpreting something, but if the energy burst that was released when Ssil Vida appeared did not affect Patir or other systems, then why did the Remnant try to replicate that energy burst on Ssil Vida in an attempt to bring back Patir?

Author: From Patir 1:
`"I only have theories for now, none of which can be confirmed. People more knowledgeable than me in the subject are examining the phenomenon right now. But look at this," she says, opening a star map on a holographic projector. "This white dot is Postverta, and we are here. When Ssil Vida first appeared, it released a strong burst of ionizing radiation, and when we were able to calculate its velocity, we found it was going much faster than the speed of light. It is gradually slowing down, so it will not go much further, but exactly the same day that the energy burst was supposed to arrive here, Patir appeared! It would be an incredible coincidence if the two events were not connected.`

Reviewer 7: But doesn't this contradict line 1872? What do you mean by 'no correlation with the systems controlling its pocket universe' in line 1872? Sorry if I'm being a bit dense here.

Author: No worries. The meaning there was that the initial burst that revealed the system to you was the one that made Patir appear, but when the Remnant later learned how to move Ssil Vida back and forth between dimensions, that function had no relation with Patir itself; that is, whether the Remnant or the player activate or deactivate Ssil Vida has no influence on Patir's appearance or disappearance, and it took some time and effort on the part of the Remnant for them to get Ssil Vida to emit the right signal again

Reviewer 7: Ah I see. Sorry, it's been a while since I've played through Remnant content. Maybe this would make it a bit clearer?

```suggestion
	`	"We knew Postverta was involved with Patir's initial arrival, and together with the team there, we discovered that the energy burst that had been released when Ssil Vida first appeared had no relation to the systems controlling its pocket universe. So, its activation or deactivation would have no effect on Patir, and we had to start looking somewhere else.`
```

	`	"The project took most of our workforce, as well as several weeks to complete, no thanks to a number of setbacks almost making us start over from scratch. But with the full power of the onboard reactors and an array of calculators rivaling that of our largest outposts, its giant antennae came back to life, and released a wave as strong as when Ssil Vida appeared. Two weeks later, Patir returned, and now you are here, safe and sound."`
	`	Finally, a tired Dusk reaches the two of you in the storage room, sitting down after greeting Chilia and you. From a brief exchange between them, you infer that he has also been told of the circumstances around your return.`
	choice
		`	"What is going to happen to Patir now?"`
			goto patir
		`	"Wait, what if we were not around Patir when it came back?"`

> 	`	"The black hole would have departed without us, and we would have been still stuck in that sector - only this time without a laboratory to support us, nor most of our consumables. The people down here perhaps could have been able to send Patir back there to pick us up, but by then we would have run out of supplies long before, and perished between the stars. We were lucky that things went the way they did, all things considered. Our research in those systems was supposed to bring us back, and yet we were just chasing rainbows, endangering us further every step we took.`

Reviewer: ```suggestion
	`	"The black hole would have departed without us, and we would have been still stuck in that sector - only this time without a laboratory to support us, nor most of our consumables. The people down here perhaps could have been able to send Patir back there to pick us up, but by then we would have run out of supplies long before, and perished between the stars. We were lucky that things went the way they did, all things considered. Our research in those systems was supposed to bring us back, and yet we were just chasing rainbows, endangering ourselves further every step we took.`
```

	"reputation: Ka'sei" = 1

event "patir mystery timer"

ship "Cloak Check"
	attributes
		"heat dissipation" 1
		"hull" 3
		"mass" 100
		"drag" 5
		"automaton" 1
		"thrust" 100
		"turn" 6000
	outfits
		"Cloak Checker"
	turret 0 0

outfit "Cloak Checker"
	category "Turrets"
	thumbnail "outfit/unknown"
	"turret mounts" -1
	weapon
		"velocity" 10
		"lifetime" 1
		"reload" 1
		"blast radius" 20
		"hull damage" 5
		"no damage scaling"
		"range override" 1000000
		"turret turn" 360

mission "Ka'het: Patir Mystery 6"
	name "Second contact"
	description "Return to <system> with Dusk and four other xenoarchaeologists to continue your exploration of the <planet>. You will need to be cloaked as you enter to avoid alerting the Ka'sei."
	source
		government "Remnant"
		attributes "remnant primary"
	destination "Builder Settlement"
	passengers 5
	cargo "Reserved space" 2
	blocked `As soon as you arrive, Dusk contacts you again. "Welcome back, <first>! We should be ready for the next Patir expedition, but we need at least five free bunks to attempt it, and at least 2 tons of cargo space for samples."`
	to offer

> 		random < 60

Author: Changed the event delay to 10-16 and removed the random

			branch "atmospheric data"
				has "atmospheric scan intact"
			`	One of the other Remnant shows you a hologram of the system, with what looks like the Builder settlement you visited earlier highlighted by a yellow glow. "After studying the data we gathered during our last visit, we may have devised a way to enter the structure undisturbed! Based on our recordings, the Fetri'sei that attacked us in the structure was gathering materials from the asteroid belt with its fellow Ka'sei, and only moved inside the dome when it spotted us."`
				goto proposal

			label "atmospheric data"
			action
				payment 300000
			`	One of the other Remnant shows you a hologram of the system, with what looks like the Builder settlement you visited earlier highlighted by a yellow light. "First of all, we have transferred <payment> to your logistics account for saving the data on the dome's atmosphere. It is only thanks to you that we were able to learn more about what kind of atmosphere the Builders may have lived in."`
			choice
				`	"Thank you."`
					goto skipdialog
				`	"Can you tell me more about that?"`

> 			`	"It seems their atmosphere may have been similar to ours - the air in the settlement we visited was almost evenly split between oxygen and argon: breathable by human standards."`

Author: Changed to `	"The air in the settlement we visited was almost evenly split between oxygen and argon, while containing very little nitrogen. Despite its differences, it is interesting to note that such a composition is breathable by human standards."`

			label proposal
			choice
				`	"What are you suggesting?"`
				`	"And?"`
			`	"Cloaking is the key. If we can stay hidden as we enter Patir and land, then the Fetri'sei should stay outside of the settlement, letting us explore further. Make sure your ship has enough fuel for the journey, however; if you drop your cloak at any time in Patir, then the Ka'sei will block our progress again."`
			`	After he finishes signing, he makes a few quick gestures toward the projector, enlarging the hologram of the Builder settlement and revealing what looks like a flight path. "You are going to land roughly where we did last time, carrying me and four other xenoarcheologists. If you accept, I should be able to allocate 100,000 credits to help you buy the necessary equipment for this mission."`
			choice
				`	"I'm in."`
					accept
				`	"What if they detect us while inside the structure?"`
				`	"Sorry, but I don't think it is a good idea to return there."`
					decline

> 			`	"It is a possibility, yes. It is also quite problematic because communication between the inside and the outside of the structure is impossible. I think the best course of action would be to place a scanner that controls the large airlocks on the upper side of the dome, the same ones the Ka'sei used last time. If I am right, it should give us around a quarter of an hour to get back in the airlock and escape."`

Reviewer: It makes sense to me - the scanner checks to make sure nothing is entering the airlocks. However, it might be improved by
```suggestion
			`	"It is a possibility, yes. It is also quite problematic because communication between the inside and the outside of the structure is impossible. I think the best course of action would be to place a scanner that keeps an eye on the large airlocks on the upper side of the dome, the ones the Ka'sei used last time. If I am right, it should give us around a quarter of an hour to get back in the airlock and escape."`
```

			choice
				`	"Alright, let's do it."`
					accept
				`	"Sorry, but I don't think it is a good idea to return there."`
					decline

	on accept
		payment 100000
		set "remnant: dusk busy"
	on abort
		clear "remnant: dusk busy"
	on fail
		clear "remnant: dusk busy"
		dialog

> 			`You have approached the Builder Settlement uncloaked, making the Ka'sei follow you, or the escort that was carrying the Remnant scientists has been destroyed. You have failed this mission.`

Author: Changed the reserved space to "instruments and samplers" and expanded the on fail message

			action
				"patir: builder seeds" ++
			`	"I agree, it feels the most interesting of the two." The two of you follow the route indicated on the map for a while before reaching the entrance: a slender, sturdy-looking pillar roughly two meters wide with a circular opening on one side. Moving closer you can see the end of a shallow ramp right beyond the door, swirling upwards inside of the column and likely reaching the top of the dome.`
			`	"If this is supposed to be the exit of the garden," Dusk says, "I fail to see where the entrance should be. We are going to have to walk up it instead."`
			choice
				`	"Alright, let's go."`
					goto ramp
				`	"I would rather keep searching for the entrance."`
			`	You and Dusk keep walking around the structure for almost half an hour to no avail; with no other choices left but going up the ramp, you give up and decide to follow it up the pillar. Going through it is simpler than it looked, as the surface of the floor is ribbed in a peculiar way so that gripping onto it becomes rather effortless. Despite this, it takes half an hour to reach the garden, and the roof of the dome still lies well above.`
				goto top

			label ramp

> 			`	You follow Dusk up along the ramp, the length of it making the ascent quite tiresome. Luckily, going through it is simple, as the surface of the floor is ribbed in a peculiar way so that gripping onto it becomes rather effortless. It still takes a while to reach the end: even so, though the garden is nowhere near the top of the dome, it takes another half an hour to reach it.`

Author: I've removed the part of the phrase before the colon entirely

				accept
	on stopover
		dialog
			`Dusk's coordinates indicate the border of a large canyon near the equator of <planet stopovers>. Unlike most cracks on the surface, this one shows signs of geological activity: you only narrowly avoid a stream ejecting from a geyser in the canyon as you pass overhead. As you land, a row of Remnant structures are clearly visible running parallel to the cliff.`
			`	The local crew assist in loading the modular laboratory split across three pallets; with them arrives a separate package, containing a surveillance satellite to be deployed near Patir while cloaked.`
	on abort
		clear "remnant: dusk busy"
	on fail
		clear "remnant: dusk busy"
		dialog
			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`

	npc save
		government "Test Dummy"
		personality unconstrained heroic nemesis
		system "Patir"
		fleet
			variant
				"Cloak Check" 10

mission "Ka'het: Patir Mystery 8B"
	landing
	invisible
	source "Babiali"
	destination "Builder Settlement"
	to offer

> 		has "Ka'het: Patir Mystery 8A: offered"

Author: What's the best way to work around that?

> 		has "Ka'het: Patir Mystery 8A: offered"

Reviewer 3: I would probably split mission 8A into two missions: one from Aventine to Babiali, and then another from Babiali to the Builder Settlement, with the `on enter "Patir"` being a part of the second mission.
Is there any reason that wouldn't work?

				`	"How do we get back?"`
				`	"Was that a hyperspace jump then?"`
			`	"I have no idea, <first>. What I can tell you for now is that we need to get a clearer idea of what we have around before deciding the next step, and there is no way to get that from here. You need to explore as far as possible in this new sector, and return once you either find a way out or run out of systems. I promise we will have a plan ready by the time you're back."`
				accept
	on enter Chanai
		dialog
			`As soon as you enter Chanai, your sensors detect a massive structure of artificial nature, apparently a station of some kind.`
	on enter Athiri
		dialog
			`As you enter the system, your sensors lose any trace of the wormhole you passed from. There doesn't seem to be a way to pass through it again from this end.`
	on visit
		dialog
			`Observing the systems you have mapped, Dusk seems disappointed. "There must be a way to reach the other systems. Please look everywhere you can for a passage; there has to be something."`

event "Patir moves"
	system Patir
		pos 10059.5 10.5
		remove haze
		add hazard "Beyond Patir Shield Eraser" 1
		remove hazard "Ember Waste Base Heat" 100

> 		remove hazard "Ember Waste Base Storm" 9000

Author: Sorry, in what way?

> 		remove hazard "Ember Waste Base Storm" 9000

Reviewer 3: ```
system "Patir"
    [...]
    hazard "Ember Waste Base Heat" 200
        to spawn
            has "patir in milky way"
    hazard "Beyond Patir Shield Erase" 1
        to spawn
            has "patir beyond milky way"
```
Or whatever conditions you want. And switch between the conditions with these events instead of making changes to the system.

> 		remove hazard "Ember Waste Base Storm" 9000

Author: Does this work with Haze as well? For now I've done it just with hazards

	planet "Builder Settlement"
		spaceport `The Remnant laboratory merges oddly well with the rest of the environment, similar in color and design to the Builder structures around here. The spaceport consists only of a flat strip of regolith near the settlement, hastily cleaned of the debris that covers most of the terrain.`

mission "Ka'het: Patir Mystery 10"
	landing
	name "Gather data"
	description "Board the Remnant satellite in orbit around <system> to learn more about the odd creatures around here."

> 	passengers 1

Author: Added

			`	After searching a large bag on the ground, one of the Remnant brings out another holographic projector. She tinkers with it for a bit until it focuses on the neutron star you visited earlier, Lathia. "This is our safest bet at learning more about these creatures," she signs. "According to your data, this system has by far the highest number and diversity of them. If you pass through the system while cloaked and use our sensors, you may be able to obtain enough information without putting yourself at risk."`
			choice
				`	"If those scanners did not work on them here, why would it be different in Lathia?"`
					goto why
				`	"What if I run out of fuel?"`
					goto fuel
				`	"What happens if this fails as well?"`
			`	"That is a distinct possibility," Dusk replies, "but we must investigate every avenue before we resort to risking personnel."`
				goto assembly

			label fuel

> 			`	"It will only take a few minutes to determine whether our equipment can pass through the exterior of these beings," Dusk replies. "I suggest filling your fuel tanks near Patir before leaving."`

Author: Fixed

				goto assembly
			label why
			`	"It is a shot in the dark," Dusk replies. "Being able to scan even just one of these beings would be a great leap forward - perhaps one without external shielding, or damaged in such a way that permits us to analyze them."`
			label assembly
			`	The Remnant begin to mount and wire all the scanning equipment on the <ship>. After a few hours, you're ready to take off once again.`
				accept
	on enter Lathia
		dialog
			`A dull hum from below the cockpit signals that the scanners have activated. You take the opportunity to observe the mesmerizing colors of the neutron star, beautiful and terrifying at the same time. After a few minutes, a beeping sound alerts you that the scan is complete, and you can return to <system>.`
	npc
		government "Magic Asteroid"

> 		personality coward fleeing harvests pacifist uninterested waiting

Author: I just copied the personalities from 12D, but most of those are not actually necessary here. I've changed it to use the personalities from the standard fleets instead, with the addition of waiting and staying.

Also, while I was at it I added mute to the personality of 12D since it was missing

		fleet
			cargo 3
			variant
				"Asteroid Large 4"
				"Asteroid Medium"
				"Asteroid Young 2"
				"Asteroid Young 1"

mission "Ka'het: Patir Mystery 12A"
	landing
	name "The Chanai station"
	description "Investigate the station in Chanai for a way to return to the Milky Way. Afterwards, return to <system>."
	passengers 5
	source "Builder Settlement"

> 	stopover "Chanai Structure"

Author: There shouldn't be any issue with splitting it, is it necesssary?

> 	stopover "Chanai Structure"

Reviewer 3: I generally think non-stopover missions are better when there isn't a benefit to using a stopover.
Having a stopover means selecting the mission doesn't immediately pan the map to the place you need to go next, for example.

			label moving
			`	The rest of the zone isn't as interesting. A series of holes in a wall attract your attention, but they prove too narrow to pass through and too long to see the end; similarly, the corridor at the end of the area is slightly smaller than your suit, preventing you from continuing. As you return back to the rest of the group, however, you notice an unusual drawing.`
			`	The canvas, attached between various carvings, shows a series of points in a line with every color of the rainbow. While the lines are near the bottom of the mural, above them all shines what looks like a red sun, sending rays of light down to each of the colorful dots and surrounding them all in a crimson halo. You pass your glove over its surface, and you notice every part is composed of a different material, smoother and more refined than the walls around it.`

> 			`	Based on your oxygen levels, you need to meet up with the Remnant team as soon as possible; you follow Dusk's instructions from the radio up and down the structure until reaching a large open area, where you find them observing a vial of samples from the structure. Behind them, you see the start of the giant barrel that composes most of the station's structure.`

Author: Changed to `	While the oxygen levels in your suit continue to lower, you follow Dusk's instructions from the radio up and down the maze of tunnels until reaching a large open area where you finally find the Remnant, busy observing a vial of samples. Behind them, you see the start of the giant barrel that composes most of the station's structure.`

			`	You exit the station and enter your ship, and see most of the scientists have already moved their samples inside your cargo hold, with around two dozen containers scattered on the floor. Soon after, you're back inside the cockpit, preparing for liftoff; discussion about the next mission will have to wait until you're in <system>.`
	on complete
		conversation
			`As you accompany the Remnant team back to the lab, they confirm Dusk's theories on the dust. From the way they speak about it, they seem to have reached a consensus on the source of the regolith, though they don't go into details.`

> 			`	The module was always cramped, but over the last few weeks, a thin layer of dust coming from outside the asteroid has covered the floor, with just the surfaces needed for the research kept clean. Tiredness plagues the faces of the scientists: they must not have slept much in the past days to keep up with the amount of incoming data, and neither have you. As you try to remember the last time you stepped outside without a suit - at least a month, most likely - your thoughts are interrupted by the arrival of Dusk, carrying on his back the last few samples.`

Author: The intended meaning was that the interior of the lab was never very nice looking, but on top of that over time it has become dusty, and generally quite messy. I've changed the beginning of the phrase to `The module was never particularly inviting` in an attempt to clarify

				goto lab
			label plan
			`	"We have been considering a close approach to one of those beings ever since we first saw them, and your discovery of the planetoid in Lathia opens up the possibility for a surface expedition. Thanks to its size, landing with the <ship> should be possible, and we could even explore the planetoid in our suits if the surface is safe. Perhaps, if these beings are truly intelligent, this could be the first contact between our two species.`
			label lab
			`	"Meet us in the lab in a few hours, and we will have everything ready for the trip. With a bit of luck, most of the instruments we used on Chanai will be a good fit for this mission as well."`
		
mission "Ka'het: Patir Mystery 12B"
	name "Satellite deployment"
	description "Deploy two probes in <stopovers> to keep an eye on incoming space rocks."
	source "Builder Settlement"
	passengers 1
	cargo "Remnant Probes" 3
	waypoint "Mitera"
	waypoint "Thepa"
	to offer

> 		has "Ka'het: Patir Mystery 12A: declined"

Author: I went with the second option, but did not test it yet

				accept
	on enter "Mitera"
		dialog
			`As the satellite leaves your ship, you watch it move further away and unfurl its solar arrays. The probe quickly disappears in the blackness of space.`
	on enter "Thepa"
		dialog
			`Dusk quickly reaches for the console, initiating the satellite deployment procedure you've become accustomed to. It doesn't take long for it to complete, and in a few minutes you're ready to leave the system once again.`

ship "Asteroid Blocker"
	attributes
		"heat dissipation" 1
		"hull" 3
		"mass" 100
		"drag" 5
		"automaton" 1
		"thrust" 100
		"turn" 6000
	outfits
		"Fuel Removal"
	turret 0 0

outfit "Fuel Removal"
	category "Turret"
	thumbnail "outfit/unknown"
	"turret mounts" -1
	weapon
		"velocity" 1000
		"lifetime" 10
		"reload" 100000
		"blast radius" 200000
		"safe"
		"no damage scaling"
		"range override" 10000000
		"homing" 4
		"tracking" 1
		"acceleration" 1.5
		"drag" 0.2
		"turn" 12
		"fuel damage" 500
		"turret turn" 360

mission "Ka'het: Patir Mystery 12C"
	landing
	name "Ka'sei hacking"

> 	description "Taking advantage of the arrival of one of the asteroids, berth with a downed Ka'sei to gather data from its internal computers."

Author: In a space-related context they are used more loosely, but docking works just as well for me so I committed the suggestion

	passengers 5

> 	cargo "Ka'het boarding payload" 3

Author: I think it makes more sense to just carry all the 5 remnant also on the previous mission, so that you're forced to have space for them already

> 	cargo "Ka'het boarding payload" 3

Author: The only sentence that seemed to require changes was `	On the ground, the whole team is outside the lab, their hands full of strange equipment and unwieldy machinery. You set your ship's ramp to deploy and quickly walk down to Dusk so you can help them carry the instruments aboard.` which became `	On the ground, the whole team rushes out of your ship toward the lab, soon returning with their hands full of strange equipment and unwieldy machinery. You quickly prepare your ship's cargo bay for the new payloads and walk down to Dusk so you can help them carry the instruments aboard.`

			label move
			`	"Inside, we will search for the memory banks of the spacecraft. After a quick selection based on the data we can scrape, we will take the few that contain information concerning our research and extract their contents. Understood?"`
			choice
				`	(Nod.)`
					goto entrance
				`	"What about me?"`
			`	"Given that you need to be able to fire as quick as possible, you cannot return to the cockpit. So, we took the liberty of creating a small terminal here to keep an eye on the situation." He points to a small desk in the cargo bay filled to the brim with Remnant machinery.`
			label entrance
			`	One by one, the scientists jump from your ship to the Fetri, moving toward a closed, circular hatch in between the two engines. As you watch them start the expedition, you sit down at the makeshift console, monitoring movements in the system as well as any changes to the slug. Though you can't clearly see what they are doing, you distinguish someone inserting a rod inside a small aperture, followed by the hatch opening right after.`

> 			`	As the crew moves beyond your line of sight and communications cease, your unease grows steadily. You split your attention between the various screens in front of you while glancing out of the airlock, and the clock on your suit's wrist keeps on ticking slowly.`

Reviewer: Just one more change.
```suggestion
			`	As the crew moves beyond your line of sight and communications cease, your unease grows steadily. You split your attention between the various screens in front of you and glancing out of the airlock. The clock on your suit's wrist keeps on ticking slowly.`
```

	gun -13 -36
	gun 13 -36
	turret 0 0
	engine -20 46.5
	engine 20 46.5
	explode "tiny explosion" 6
	explode "small explosion" 10
	explode "medium explosion" 18
	explode "large explosion" 10
	"final explode" "final explosion small"
	description "The Fetri is a self replicating probe, built to harvest iridium away from the Graveyard and transport it back to the Builders."

ship "Asteroid Large 1" "Asteroid Large (Patir 12A)"
	sprite "ship/asteroid adult"
	attributes
		category "Transport"
		"cost" 1350000000
		"hull" 965612
		"required crew" 1
		"bunks" 1
		"mass" 130
		"drag" 4.1
		"heat dissipation" .7
		"energy capacity" 1000
		"energy generation" 10
		"fuel capacity" 250
		"fuel generation" -1
		"cargo space" 50
		"outfit space" 390
		"weapon capacity" 150
		"engine capacity" 95
		weapon
			"blast radius" 80
			"shield damage" 800
			"hull damage" 400
			"hit force" 1200
	outfits
		"Asteroid Anti-Missile"

		"Asteroid Jump Drive"
		
	engine 0 0
		zoom 1
		angle 0
		over
	turret 0 0 "Asteroid Anti-Missile"
	explode "tiny explosion" 10
	explode "small explosion" 25
	explode "medium explosion" 25
	explode "large explosion" 10
	"final explode" "final explosion medium"

> mission "Ka'het: Patir Mystery 13B"

Reviewer 3: This mission has no passengers.

Author: Added

	landing
	name "Return to <planet>"
	description "Now that the Patir black hole appears to have returned to the Milky Way, return to <planet> together with Dusk."
	source "Builder Settlement"
	destination "Aventine"
	to offer
		has "Ka'het: Patir Mystery 12C: done"
	on offer

> 		event "Patir returns" 0

Reviewer 3: I think it might have been this mission where Derpy asked to move the event application into the conversation because this one actually has choices before Patir moves.

Author: Done

			`	When the scientists finish unloading their cargo, Dusk calls you with the communicator. Now that the immediate danger is over, a hint of frustration creeps into his voice.`
			`	"The cells we have taken from the Fetri are only a few out of multiple hundreds. We did our best to choose those that were recorded at times when those beings were present, but we will still need some luck to get the results we wanted.`
			`	"Our calculators have been crunching numbers for hours now, ever since we cracked open the first canister. By the time you have reached the lab, we should have the analysis ready." You close the hatch behind you to join the rest of the crew.`
			`	As the asteroid continues its nearly imperceptible spin, Patir emerges from behind the giant dome, allowing its rays to pass through. You take the last steps towards the outpost with the sunshade lowered and walk into the lab once more.`

> 			`	The module was always cramped, but over the last few weeks, a thin layer of dust coming from outside the asteroid has covered the floor, with just the surfaces needed for the research kept clean. Tiredness plagues the faces of the scientists: they must not have slept much in the past days to keep up with the amount of incoming data, and neither have you. As you try to remember the last time you stepped outside without a suit - at least a month, most likely - your stream of thoughts is interrupted by the arrival of Dusk, carrying in his hands a ream of papers.`

Reviewer 3: Same thing about the first sentence as the other place.

				`	"Are we back in the Milky Way?"`
				`	"How long was that?"`
					goto time
				`	"Where are we now?"`
			`	"I cannot say yet," Dusk replies hurriedly. "Quick, put your EVA suit back on. We need to depart as soon as possible, all of us together. I do not know how much time we have." You hastily connect your helmet back to the life support and follow them out of the airlock, trying to keep up as best as you can.`
				goto depart

			label time
			`	"Around four minutes," Dusk replies hurriedly. "But we will talk about that another time. We need to depart as soon as possible, all of us together. I do not know how much time we have." You hastily connect your helmet back to the life support and follow them out of the airlock, trying to keep up as best as you can.`
			label depart
			`	It takes only a few minutes to launch the <ship>, the six of you standing in the cockpit with trepidation. "We will take care of the necessary analysis while you bring us to orbit. Hopefully we have been sent to the Milky Way, and not to yet another unreachable galaxy."`
				launch
	on enter
		conversation "end of Patir 2 - on enter"

	on complete
		payment 1570000
		clear "remnant: dusk busy"

> 		clear "remnant: dusk busy"

Reviewer 3: Don't need to clear the same condition twice. Did you mean to clear something else?
```suggestion
```

Author: There's nothing else to be cleared, so it's just a remnant of some changes done recently it would seem

		event "patir mystery 2 timer" 45
		set "initial patir exploration"
		conversation "end of Patir 2 - on complete"

event "Patir returns"
	system Patir
		pos -38 553
		haze _menu/haze-red
		remove hazard "Beyond Patir Shield Eraser" 1
		add hazard "Ember Waste Base Heat" 100
		add hazard "Ember Waste Base Storm" 9000

> mission "Ka'het: Patir Mystery 13A"

Reviewer 3: Can this be listed before 13B?
Also, passengers?

Author: Swapped

			`	You move the airlock's heavy door to pass through, and wait for the room to be re-pressurized before walking in. By the time the hatch unlocks, however, the Remnant are gone, hastily leaving their bags on the floor.`
			`	As you take a few steps inside the lab, you turn around to find them all glued to the windows with an awestruck gaze. The sky has again turned black, and a few dim lines are already starting to appear in the distance.`
			`	While stars and galaxies zoom in the night sky, a few of the Remnant run around the lab, turning on various instruments. However, most of them remain with their faces stuck to the glass. The lights start converging toward a single point, forming a large, but rapidly shrinking mass in the background.`
			`	The stars begin to slow down before settling into their final positions. After all the time you have spent here, it isn't difficult to recognize that the sky has changed again. Next to you, the rest of the team is already analyzing the data gathered; Dusk seems to be writing down something from a chronometer.`
			action

> 				event "Patir returns" 0

Reviewer 3: > Derpy asked to put this here so that the map will show the right position before and after patir moves

Has this been tested? Since there's no choice prior to this point in the conversation, I'm fairly sure it will apply this event immediately upon the conversation appearing, which is the same as if it were placed directly in the trigger, outside of the conversation.
If you were to put the action after the choice, then it would only apply after that choice has been made, but Patir has already moved before we reach the choice.

Author: > Has this been tested?

No. Moved back outside of the conversation

				launch
	on enter
		conversation "end of Patir 2 - on enter"
	on complete
		payment 1570000
		clear "remnant: dusk busy"
		set "initial patir exploration"
		event "patir mystery 2 timer" 45
		conversation "end of Patir 2 - on complete"

conversation "end of Patir 2 - on enter"
	`The cockpit has become quite chaotic: the instruments the team brought with them keep buzzing and beeping. After shutting down the main engines, someone taps on your shoulder to call your attention. "<first>, could you pitch the ship down a few degrees?" Dusk asks, with a calmer look than during liftoff. You reply with a nod, and move the stick forward slightly.`
	`	"There! Do you see it?" he continues, pointing at a spot in the sky with his hand. Between the mass of stars in front of you, a pattern leaps to the eye: two sets of three stars, right next to each other, forming a prism-like structure. "We are back."`

conversation "end of Patir 2 - on complete"

> 	`A large crowd has quickly gathered around your landing pad, eagerly waiting your arrival. After all this time between the stars, setting foot on a planet with an actual atmosphere is incredibly refreshing. Ignoring the cold weather for a moment, you take the biggest breath you can to make up for the past weeks of recycled air.`

Reviewer 3: ```suggestion
	`A large crowd has quickly gathered around your landing pad, eagerly awaiting your arrival. After all this time between the stars, setting foot on a planet with an actual atmosphere is incredibly refreshing. Ignoring the cold weather for a moment, you take the biggest breath you can to make up for the past weeks of recycled air.`
```
Or, "waiting for", but I think "awaiting" is more natural.

Reviewer: ```suggestion
	`A large crowd quickly gathers around your landing pad, eagerly awaiting your arrival. After all this time between the stars, setting foot on a planet with an actual atmosphere is incredibly refreshing. Ignoring the cold weather for a moment, you take the biggest breath you can to make up for the past weeks of recycled air.`
```

	source "Aventine"
	destination "Babiali"
	passengers 5
	blocked `Dusk contacts you soon after you reach the spaceport. "<first>, we will need at least five free bunks for the mission we are preparing."`
	to offer
		has "Ka'het: Patir Mystery 7: done"
	on offer
		conversation
			`It doesn't take long for you to see Dusk; he's waiting for you under a canopy next to your landing pad, right at the edge of Aventine's starport. "We are almost ready to go," he says, "but I have run into a problem. My plan for our next mission was to have a repeat of the last expedition, but this time carrying a temporary laboratory to the Builder site. I was able to gather most of the instruments I was looking for in the local scientific facilities, but none of the mobile laboratories I am authorized to requisition by default were in good enough condition to fly.`
			`	"Because of that, before we can start our mission, we will need to make a stopover at <stopovers> - I was able to convince the prefects on Viminal to reallocate one of the outposts that they had just delivered there to our mission, given it has higher priority than the mining expeditions they are setting up."`

> 			`	You help Dusk transport the chosen instruments to your ship; you also make sure to leave enough space for the facilities you're going to take from <planet stopovers>. As you're walking back to the ship's controls, Dusk reminds you to bring as many ramscoops with you as possible - you're going to need them in order to complete the mission plan while cloaked.`

Author: Added `"In the meanwhile, make sure to prepare a large enough freighter for takeoff - chances are we will need at least a hundred tons of free space in the cargo hold for the complete payload."` at the end of the previous mission. I feel like it's more interesting to keep the requirements high enough, especially with the cloaking requirement.

					goto fuel
				`	"What happens if this fails as well?"`
			`	"That is a distinct possibility," Dusk replies, "but we must investigate every avenue before we resort to risking personnel."`
				goto assembly

			label fuel
			`	"It will only take a few minutes to determine whether our equipment can see through the exterior of these beings," Dusk replies. "I suggest filling your fuel tanks near Patir before leaving."`
				goto assembly
			label why
			`	"It is a shot in the dark," Dusk replies. "Being able to scan even just one of these beings would be a great leap forward - perhaps one without external shielding, or damaged in such a way that permits us to analyze them."`
			label assembly
			`	The Remnant begin to mount and wire all the scanning equipment on the <ship>. After a few hours, you're ready to take off once again.`
				accept
	on enter Lathia
		dialog
			`A dull hum from below the cockpit signals that the scanners have activated. You take the opportunity to observe the mesmerizing colors of the neutron star, beautiful and terrifying at the same time. After a few minutes, a beeping sound alerts you that the scan is complete, and you can return to <system>.`
	npc

> 		government "Magic Asteroid"

Reviewer 6: ```suggestion
		government "Magic Asteroid"
		system "Lathia"
```
